<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/1999/REC-html401-19991224/loose.dtd">
<html><head><title>Report for PS218_2h 1610970986809 [GSEA]</title><link href='xtools.css' rel='stylesheet'><link href='http://www.gsea-msigdb.org/gsea/images/icon_16x16.png' rel='shortcut icon'></head><body><div class='richTable'><table cols='11' border='1'><th class='richTable'></th><th class='richTable'>GS<br> follow link to MSigDB</th><th class='richTable'>GS DETAILS</th><th class='richTable'>SIZE</th><th class='richTable'>ES</th><th class='richTable'>NES</th><th class='richTable'>NOM p-val</th><th class='richTable'>FDR q-val</th><th class='richTable'>FWER p-val</th><th class='richTable'>RANK AT MAX</th><th class='richTable'>LEADING EDGE</th><tr><td class='lessen'>1</td><td>20.2.4 STRESS.ABIOTIC.TOUCH/WOUNDING</td><td><a href='20.2.4 STRESS.ABIOTIC.TOUCH/WOUNDING.html'>Details ...</a></td><td>27</td><td>0.88</td><td>2.23</td><td>0.000</td><td>0.000</td><td>0.000</td><td>1204</td><td>tags=52%, list=2%, signal=53%</td></tr><tr><td class='lessen'>2</td><td>20.1.7.6.1 STRESS.BIOTIC.PR-PROTEINS.PR6 (PROTEINASE INHIBITORS).TRYPSIN INHIBITOR</td><td><a href='20.1.7.6.1 STRESS.BIOTIC.PR-PROTEINS.PR6 (PROTEINASE INHIBITORS).TRYPSIN INHIBITOR.html'>Details ...</a></td><td>25</td><td>0.75</td><td>1.86</td><td>0.000</td><td>0.041</td><td>0.039</td><td>1517</td><td>tags=44%, list=3%, signal=45%</td></tr><tr><td class='lessen'>3</td><td>20.1.7.6 STRESS.BIOTIC.PR-PROTEINS.PR6 (PROTEINASE INHIBITORS)</td><td><a href='20.1.7.6 STRESS.BIOTIC.PR-PROTEINS.PR6 (PROTEINASE INHIBITORS).html'>Details ...</a></td><td>55</td><td>0.60</td><td>1.78</td><td>0.000</td><td>0.102</td><td>0.133</td><td>1929</td><td>tags=24%, list=4%, signal=25%</td></tr><tr><td class='lessen'>4</td><td>29.5.2 PROTEIN.DEGRADATION.AUTOPHAGY</td><td><a href='29.5.2 PROTEIN.DEGRADATION.AUTOPHAGY.html'>Details ...</a></td><td>22</td><td>0.68</td><td>1.65</td><td>0.007</td><td>0.189</td><td>0.302</td><td>7848</td><td>tags=68%, list=16%, signal=81%</td></tr><tr><td class='lessen'>5</td><td>27.3.36 RNA.REGULATION OF TRANSCRIPTION.ARGONAUTE TRANSCRIPTION FACTOR FAMILY</td><td><a href='27.3.36 RNA.REGULATION OF TRANSCRIPTION.ARGONAUTE TRANSCRIPTION FACTOR FAMILY.html'>Details ...</a></td><td>18</td><td>0.67</td><td>1.55</td><td>0.031</td><td>0.313</td><td>0.520</td><td>4972</td><td>tags=33%, list=10%, signal=37%</td></tr><tr><td class='lessen'>6</td><td>27.3.23 RNA.REGULATION OF TRANSCRIPTION.HEAT-SHOCK TRANSCRIPTION FACTOR FAMILY (HSF)</td><td><a href='27.3.23 RNA.REGULATION OF TRANSCRIPTION.HEAT-SHOCK TRANSCRIPTION FACTOR FAMILY (HSF).html'>Details ...</a></td><td>33</td><td>0.56</td><td>1.49</td><td>0.026</td><td>0.390</td><td>0.649</td><td>2350</td><td>tags=27%, list=5%, signal=29%</td></tr><tr><td class='lessen'>7</td><td>13.1.5 AMINO ACID METABOLISM.SYNTHESIS.SERINE-GLYCINE-CYSTEINE GROUP</td><td><a href='13.1.5 AMINO ACID METABOLISM.SYNTHESIS.SERINE-GLYCINE-CYSTEINE GROUP.html'>Details ...</a></td><td>39</td><td>0.54</td><td>1.48</td><td>0.028</td><td>0.359</td><td>0.675</td><td>4035</td><td>tags=26%, list=8%, signal=28%</td></tr><tr><td class='lessen'>8</td><td>29.5.11.4.3.1 PROTEIN.DEGRADATION.UBIQUITIN.E3.SCF.SKP</td><td><a href='29.5.11.4.3.1 PROTEIN.DEGRADATION.UBIQUITIN.E3.SCF.SKP.html'>Details ...</a></td><td>56</td><td>0.49</td><td>1.40</td><td>0.000</td><td>0.484</td><td>0.828</td><td>6059</td><td>tags=9%, list=12%, signal=10%</td></tr><tr><td class='lessen'>9</td><td>29.5.11.4.3.3 PROTEIN.DEGRADATION.UBIQUITIN.E3.SCF.CULLIN</td><td><a href='29.5.11.4.3.3 PROTEIN.DEGRADATION.UBIQUITIN.E3.SCF.CULLIN.html'>Details ...</a></td><td>36</td><td>0.51</td><td>1.40</td><td>0.010</td><td>0.445</td><td>0.836</td><td>3708</td><td>tags=25%, list=8%, signal=27%</td></tr><tr><td class='lessen'>10</td><td>26.16 MISC.MYROSINASES-LECTIN-JACALIN</td><td><a href='26.16 MISC.MYROSINASES-LECTIN-JACALIN.html'>Details ...</a></td><td>33</td><td>0.52</td><td>1.39</td><td>0.048</td><td>0.425</td><td>0.854</td><td>4542</td><td>tags=24%, list=9%, signal=27%</td></tr><tr><td class='lessen'>11</td><td>29.5.11.3 PROTEIN.DEGRADATION.UBIQUITIN.E2</td><td><a href='29.5.11.3 PROTEIN.DEGRADATION.UBIQUITIN.E2.html'>Details ...</a></td><td>58</td><td>0.48</td><td>1.38</td><td>0.032</td><td>0.405</td><td>0.870</td><td>8904</td><td>tags=69%, list=18%, signal=84%</td></tr><tr><td class='lessen'>12</td><td>11.1.15 LIPID METABOLISM.FA SYNTHESIS AND FA ELONGATION.ACP DESATURASE</td><td><a href='11.1.15 LIPID METABOLISM.FA SYNTHESIS AND FA ELONGATION.ACP DESATURASE.html'>Details ...</a></td><td>17</td><td>0.59</td><td>1.33</td><td>0.125</td><td>0.490</td><td>0.936</td><td>2250</td><td>tags=12%, list=5%, signal=12%</td></tr><tr><td class='lessen'>13</td><td>13.1.5.3 AMINO ACID METABOLISM.SYNTHESIS.SERINE-GLYCINE-CYSTEINE GROUP.CYSTEINE</td><td><a href='13.1.5.3 AMINO ACID METABOLISM.SYNTHESIS.SERINE-GLYCINE-CYSTEINE GROUP.CYSTEINE.html'>Details ...</a></td><td>32</td><td>0.51</td><td>1.33</td><td>0.068</td><td>0.458</td><td>0.938</td><td>4035</td><td>tags=28%, list=8%, signal=31%</td></tr><tr><td class='lessen'>14</td><td>26.9 MISC.GLUTATHIONE S TRANSFERASES</td><td><a href='26.9 MISC.GLUTATHIONE S TRANSFERASES.html'>Details ...</a></td><td>90</td><td>0.41</td><td>1.31</td><td>0.000</td><td>0.476</td><td>0.958</td><td>4187</td><td>tags=27%, list=8%, signal=29%</td></tr><tr><td class='lessen'>15</td><td>27.3.32 RNA.REGULATION OF TRANSCRIPTION.WRKY DOMAIN TRANSCRIPTION FACTOR FAMILY</td><td><a href='27.3.32 RNA.REGULATION OF TRANSCRIPTION.WRKY DOMAIN TRANSCRIPTION FACTOR FAMILY.html'>Details ...</a></td><td>97</td><td>0.39</td><td>1.24</td><td>0.033</td><td>0.652</td><td>0.993</td><td>2573</td><td>tags=25%, list=5%, signal=26%</td></tr><tr><td class='lessen'>16</td><td>11.8.1 LIPID METABOLISM.EXOTICS (STEROIDS, SQUALENE ETC).SPHINGOLIPIDS</td><td><a href='11.8.1 LIPID METABOLISM.EXOTICS (STEROIDS, SQUALENE ETC).SPHINGOLIPIDS.html'>Details ...</a></td><td>43</td><td>0.44</td><td>1.23</td><td>0.068</td><td>0.624</td><td>0.994</td><td>6062</td><td>tags=37%, list=12%, signal=42%</td></tr><tr><td class='lessen'>17</td><td>4.3 GLYCOLYSIS.UNCLEAR/DUALLY TARGETED</td><td><a href='4.3 GLYCOLYSIS.UNCLEAR/DUALLY TARGETED.html'>Details ...</a></td><td>16</td><td>0.55</td><td>1.22</td><td>0.164</td><td>0.631</td><td>0.995</td><td>5805</td><td>tags=31%, list=12%, signal=35%</td></tr><tr><td class='lessen'>18</td><td>13.1.1 AMINO ACID METABOLISM.SYNTHESIS.CENTRAL AMINO ACID METABOLISM</td><td><a href='13.1.1 AMINO ACID METABOLISM.SYNTHESIS.CENTRAL AMINO ACID METABOLISM.html'>Details ...</a></td><td>24</td><td>0.50</td><td>1.20</td><td>0.194</td><td>0.651</td><td>0.997</td><td>4896</td><td>tags=46%, list=10%, signal=51%</td></tr><tr><td class='lessen'>19</td><td>3.4 MINOR CHO METABOLISM.MYO-INOSITOL</td><td><a href='3.4 MINOR CHO METABOLISM.MYO-INOSITOL.html'>Details ...</a></td><td>19</td><td>0.52</td><td>1.19</td><td>0.183</td><td>0.644</td><td>0.997</td><td>1997</td><td>tags=16%, list=4%, signal=16%</td></tr><tr><td class='lessen'>20</td><td>5 FERMENTATION</td><td><a href='5 FERMENTATION.html'>Details ...</a></td><td>44</td><td>0.42</td><td>1.18</td><td>0.141</td><td>0.662</td><td>0.999</td><td>1079</td><td>tags=14%, list=2%, signal=14%</td></tr><tr><td class='lessen'>21</td><td>15 METAL HANDLING</td><td><a href='15 METAL HANDLING.html'>Details ...</a></td><td>145</td><td>0.36</td><td>1.16</td><td>0.000</td><td>0.689</td><td>0.999</td><td>3804</td><td>tags=21%, list=8%, signal=22%</td></tr><tr><td class='lessen'>22</td><td>27.1.3 RNA.PROCESSING.3' END PROCESSING</td><td><a href="27.1.3 RNA.PROCESSING.3' END PROCESSING.html">Details ...</a></td><td>41</td><td>0.41</td><td>1.16</td><td>0.173</td><td>0.675</td><td>0.999</td><td>7242</td><td>tags=37%, list=15%, signal=43%</td></tr><tr><td class='lessen'>23</td><td>15.2 METAL HANDLING.BINDING, CHELATION AND STORAGE</td><td><a href='15.2 METAL HANDLING.BINDING, CHELATION AND STORAGE.html'>Details ...</a></td><td>117</td><td>0.34</td><td>1.12</td><td>0.056</td><td>0.760</td><td>0.999</td><td>3804</td><td>tags=19%, list=8%, signal=20%</td></tr><tr><td class='lessen'>24</td><td>23.3 NUCLEOTIDE METABOLISM.SALVAGE</td><td><a href='23.3 NUCLEOTIDE METABOLISM.SALVAGE.html'>Details ...</a></td><td>60</td><td>0.38</td><td>1.12</td><td>0.200</td><td>0.741</td><td>0.999</td><td>4866</td><td>tags=23%, list=10%, signal=26%</td></tr><tr><td class='lessen'>25</td><td>30.11.1 SIGNALLING.LIGHT.COP9 SIGNALOSOME</td><td><a href='30.11.1 SIGNALLING.LIGHT.COP9 SIGNALOSOME.html'>Details ...</a></td><td>22</td><td>0.48</td><td>1.12</td><td>0.239</td><td>0.715</td><td>0.999</td><td>3276</td><td>tags=27%, list=7%, signal=29%</td></tr><tr><td class='lessen'>26</td><td>26.17 MISC.DYNAMIN</td><td><a href='26.17 MISC.DYNAMIN.html'>Details ...</a></td><td>29</td><td>0.41</td><td>1.09</td><td>0.262</td><td>0.817</td><td>0.999</td><td>4591</td><td>tags=28%, list=9%, signal=30%</td></tr><tr><td class='lessen'>27</td><td>28.2 DNA.REPAIR</td><td><a href='28.2 DNA.REPAIR.html'>Details ...</a></td><td>138</td><td>0.33</td><td>1.08</td><td>0.143</td><td>0.797</td><td>0.999</td><td>4743</td><td>tags=26%, list=10%, signal=29%</td></tr><tr><td class='lessen'>28</td><td>13.2.4 AMINO ACID METABOLISM.DEGRADATION.BRANCHED CHAIN GROUP</td><td><a href='13.2.4 AMINO ACID METABOLISM.DEGRADATION.BRANCHED CHAIN GROUP.html'>Details ...</a></td><td>16</td><td>0.50</td><td>1.08</td><td>0.320</td><td>0.800</td><td>1.000</td><td>7093</td><td>tags=50%, list=14%, signal=58%</td></tr><tr><td class='lessen'>29</td><td>27.3.16 RNA.REGULATION OF TRANSCRIPTION.CCAAT BOX BINDING FACTOR FAMILY, HAP5</td><td><a href='27.3.16 RNA.REGULATION OF TRANSCRIPTION.CCAAT BOX BINDING FACTOR FAMILY, HAP5.html'>Details ...</a></td><td>17</td><td>0.48</td><td>1.07</td><td>0.308</td><td>0.792</td><td>1.000</td><td>5955</td><td>tags=35%, list=12%, signal=40%</td></tr><tr><td class='lessen'>30</td><td>27.3.7 RNA.REGULATION OF TRANSCRIPTION.C2C2(ZN) CONSTANS-LIKE ZINC FINGER FAMILY (CO-LIKE)</td><td><a href='27.3.7 RNA.REGULATION OF TRANSCRIPTION.C2C2(ZN) CONSTANS-LIKE ZINC FINGER FAMILY (CO-LIKE).html'>Details ...</a></td><td>45</td><td>0.37</td><td>1.06</td><td>0.301</td><td>0.800</td><td>1.000</td><td>1775</td><td>tags=16%, list=4%, signal=16%</td></tr><tr><td class='lessen'>31</td><td>27.1.1 RNA.PROCESSING.SPLICING</td><td><a href='27.1.1 RNA.PROCESSING.SPLICING.html'>Details ...</a></td><td>88</td><td>0.34</td><td>1.06</td><td>0.300</td><td>0.793</td><td>1.000</td><td>8580</td><td>tags=45%, list=17%, signal=55%</td></tr><tr><td class='lessen'>32</td><td>29.2.1.2.1 PROTEIN.SYNTHESIS.RIBOSOMAL PROTEIN.EUKARYOTIC.40S SUBUNIT</td><td><a href='29.2.1.2.1 PROTEIN.SYNTHESIS.RIBOSOMAL PROTEIN.EUKARYOTIC.40S SUBUNIT.html'>Details ...</a></td><td>123</td><td>0.32</td><td>1.05</td><td>0.158</td><td>0.800</td><td>1.000</td><td>9657</td><td>tags=43%, list=20%, signal=53%</td></tr><tr><td class='lessen'>33</td><td>27.3.67 RNA.REGULATION OF TRANSCRIPTION.PUTATIVE TRANSCRIPTION REGULATOR</td><td><a href='27.3.67 RNA.REGULATION OF TRANSCRIPTION.PUTATIVE TRANSCRIPTION REGULATOR.html'>Details ...</a></td><td>217</td><td>0.30</td><td>1.05</td><td>0.200</td><td>0.781</td><td>1.000</td><td>5605</td><td>tags=29%, list=11%, signal=32%</td></tr><tr><td class='lessen'>34</td><td>27.3.59 RNA.REGULATION OF TRANSCRIPTION.METHYL BINDING DOMAIN PROTEINS</td><td><a href='27.3.59 RNA.REGULATION OF TRANSCRIPTION.METHYL BINDING DOMAIN PROTEINS.html'>Details ...</a></td><td>26</td><td>0.42</td><td>1.03</td><td>0.433</td><td>0.846</td><td>1.000</td><td>7758</td><td>tags=27%, list=16%, signal=32%</td></tr><tr><td class='lessen'>35</td><td>29.5.11.1 PROTEIN.DEGRADATION.UBIQUITIN.UBIQUITIN</td><td><a href='29.5.11.1 PROTEIN.DEGRADATION.UBIQUITIN.UBIQUITIN.html'>Details ...</a></td><td>54</td><td>0.36</td><td>1.02</td><td>0.343</td><td>0.834</td><td>1.000</td><td>8225</td><td>tags=41%, list=17%, signal=49%</td></tr><tr><td class='lessen'>36</td><td>34.1.1 TRANSPORT.P- AND V-ATPASES.H+-TRANSPORTING TWO-SECTOR ATPASE</td><td><a href='34.1.1 TRANSPORT.P- AND V-ATPASES.H+-TRANSPORTING TWO-SECTOR ATPASE.html'>Details ...</a></td><td>20</td><td>0.44</td><td>1.02</td><td>0.414</td><td>0.818</td><td>1.000</td><td>2441</td><td>tags=10%, list=5%, signal=11%</td></tr><tr><td class='lessen'>37</td><td>27.3.57 RNA.REGULATION OF TRANSCRIPTION.JUMONJI FAMILY</td><td><a href='27.3.57 RNA.REGULATION OF TRANSCRIPTION.JUMONJI FAMILY.html'>Details ...</a></td><td>32</td><td>0.38</td><td>0.99</td><td>0.415</td><td>0.918</td><td>1.000</td><td>5615</td><td>tags=25%, list=11%, signal=28%</td></tr><tr><td class='lessen'>38</td><td>27.3.49 RNA.REGULATION OF TRANSCRIPTION.GEBP LIKE</td><td><a href='27.3.49 RNA.REGULATION OF TRANSCRIPTION.GEBP LIKE.html'>Details ...</a></td><td>15</td><td>0.44</td><td>0.97</td><td>0.453</td><td>0.971</td><td>1.000</td><td>6663</td><td>tags=33%, list=14%, signal=39%</td></tr><tr><td class='lessen'>39</td><td>27.3.26 RNA.REGULATION OF TRANSCRIPTION.MYB-RELATED TRANSCRIPTION FACTOR FAMILY</td><td><a href='27.3.26 RNA.REGULATION OF TRANSCRIPTION.MYB-RELATED TRANSCRIPTION FACTOR FAMILY.html'>Details ...</a></td><td>30</td><td>0.36</td><td>0.95</td><td>0.520</td><td>1.000</td><td>1.000</td><td>1824</td><td>tags=20%, list=4%, signal=21%</td></tr><tr><td class='lessen'>40</td><td>26.24 MISC.GCN5-RELATED N-ACETYLTRANSFERASE</td><td><a href='26.24 MISC.GCN5-RELATED N-ACETYLTRANSFERASE.html'>Details ...</a></td><td>58</td><td>0.32</td><td>0.94</td><td>0.621</td><td>1.000</td><td>1.000</td><td>3692</td><td>tags=22%, list=7%, signal=24%</td></tr><tr><td class='lessen'>41</td><td>23 NUCLEOTIDE METABOLISM</td><td><a href='23 NUCLEOTIDE METABOLISM.html'>Details ...</a></td><td>216</td><td>0.28</td><td>0.94</td><td>1.000</td><td>1.000</td><td>1.000</td><td>5345</td><td>tags=24%, list=11%, signal=27%</td></tr><tr><td class='lessen'>42</td><td>27.3.64 RNA.REGULATION OF TRANSCRIPTION.PHOR1</td><td><a href='27.3.64 RNA.REGULATION OF TRANSCRIPTION.PHOR1.html'>Details ...</a></td><td>21</td><td>0.39</td><td>0.93</td><td>0.545</td><td>1.000</td><td>1.000</td><td>2128</td><td>tags=14%, list=4%, signal=15%</td></tr><tr><td class='lessen'>43</td><td>9.1 MITOCHONDRIAL ELECTRON TRANSPORT / ATP SYNTHESIS.NADH-DH (TYPE I)</td><td><a href='9.1 MITOCHONDRIAL ELECTRON TRANSPORT / ATP SYNTHESIS.NADH-DH (TYPE I).html'>Details ...</a></td><td>74</td><td>0.30</td><td>0.92</td><td>0.674</td><td>1.000</td><td>1.000</td><td>8794</td><td>tags=30%, list=18%, signal=36%</td></tr><tr><td class='lessen'>44</td><td>1.1.5 PS.LIGHTREACTION.OTHER ELECTRON CARRIER (OX/RED)</td><td><a href='1.1.5 PS.LIGHTREACTION.OTHER ELECTRON CARRIER (OX/RED).html'>Details ...</a></td><td>21</td><td>0.38</td><td>0.91</td><td>0.594</td><td>1.000</td><td>1.000</td><td>2629</td><td>tags=24%, list=5%, signal=25%</td></tr><tr><td class='lessen'>45</td><td>27.3.14 RNA.REGULATION OF TRANSCRIPTION.CCAAT BOX BINDING FACTOR FAMILY, HAP2</td><td><a href='27.3.14 RNA.REGULATION OF TRANSCRIPTION.CCAAT BOX BINDING FACTOR FAMILY, HAP2.html'>Details ...</a></td><td>15</td><td>0.40</td><td>0.91</td><td>0.573</td><td>1.000</td><td>1.000</td><td>5878</td><td>tags=33%, list=12%, signal=38%</td></tr><tr><td class='lessen'>46</td><td>20.1.3.1 STRESS.BIOTIC.SIGNALLING.MLO-LIKE</td><td><a href='20.1.3.1 STRESS.BIOTIC.SIGNALLING.MLO-LIKE.html'>Details ...</a></td><td>21</td><td>0.37</td><td>0.90</td><td>0.595</td><td>1.000</td><td>1.000</td><td>4389</td><td>tags=29%, list=9%, signal=31%</td></tr><tr><td class='lessen'>47</td><td>23.3.3 NUCLEOTIDE METABOLISM.SALVAGE.NUDIX HYDROLASES</td><td><a href='23.3.3 NUCLEOTIDE METABOLISM.SALVAGE.NUDIX HYDROLASES.html'>Details ...</a></td><td>36</td><td>0.34</td><td>0.90</td><td>0.655</td><td>1.000</td><td>1.000</td><td>1801</td><td>tags=14%, list=4%, signal=14%</td></tr><tr><td class='lessen'>48</td><td>26.19 MISC.PLASTOCYANIN-LIKE</td><td><a href='26.19 MISC.PLASTOCYANIN-LIKE.html'>Details ...</a></td><td>44</td><td>0.32</td><td>0.89</td><td>0.710</td><td>1.000</td><td>1.000</td><td>3426</td><td>tags=16%, list=7%, signal=17%</td></tr><tr><td class='lessen'>49</td><td>27.3.30 RNA.REGULATION OF TRANSCRIPTION.TRIPLE-HELIX TRANSCRIPTION FACTOR FAMILY (TRIHELIX)</td><td><a href='27.3.30 RNA.REGULATION OF TRANSCRIPTION.TRIPLE-HELIX TRANSCRIPTION FACTOR FAMILY (TRIHELIX).html'>Details ...</a></td><td>32</td><td>0.33</td><td>0.87</td><td>0.730</td><td>1.000</td><td>1.000</td><td>5658</td><td>tags=31%, list=11%, signal=35%</td></tr><tr><td class='lessen'>50</td><td>13.2 AMINO ACID METABOLISM.DEGRADATION</td><td><a href='13.2 AMINO ACID METABOLISM.DEGRADATION.html'>Details ...</a></td><td>124</td><td>0.27</td><td>0.87</td><td>0.933</td><td>1.000</td><td>1.000</td><td>5943</td><td>tags=21%, list=12%, signal=24%</td></tr><tr><td class='lessen'>51</td><td>13.2.7 AMINO ACID METABOLISM.DEGRADATION.HISTIDINE</td><td></td><td>33</td><td>0.34</td><td>0.87</td><td>0.710</td><td>1.000</td><td>1.000</td><td>149</td><td>tags=3%, list=0%, signal=3%</td></tr><tr><td class='lessen'>52</td><td>18.4 CO-FACTOR AND VITAMINE METABOLISM.PANTOTHENATE</td><td></td><td>16</td><td>0.39</td><td>0.84</td><td>0.753</td><td>1.000</td><td>1.000</td><td>5819</td><td>tags=38%, list=12%, signal=43%</td></tr><tr><td class='lessen'>53</td><td>23.5 NUCLEOTIDE METABOLISM.DEOXYNUCLEOTIDE METABOLISM</td><td></td><td>17</td><td>0.37</td><td>0.83</td><td>0.686</td><td>1.000</td><td>1.000</td><td>4125</td><td>tags=41%, list=8%, signal=45%</td></tr><tr><td class='lessen'>54</td><td>20.1.3 STRESS.BIOTIC.SIGNALLING</td><td></td><td>57</td><td>0.28</td><td>0.83</td><td>0.971</td><td>1.000</td><td>1.000</td><td>6996</td><td>tags=28%, list=14%, signal=33%</td></tr><tr><td class='lessen'>55</td><td>21.2.2 REDOX.ASCORBATE AND GLUTATHIONE.GLUTATHIONE</td><td></td><td>15</td><td>0.38</td><td>0.83</td><td>0.713</td><td>1.000</td><td>1.000</td><td>8140</td><td>tags=53%, list=17%, signal=64%</td></tr><tr><td class='lessen'>56</td><td>9.1.2 MITOCHONDRIAL ELECTRON TRANSPORT / ATP SYNTHESIS.NADH-DH (TYPE I).LOCALISATION NOT CLEAR</td><td></td><td>53</td><td>0.29</td><td>0.82</td><td>0.973</td><td>1.000</td><td>1.000</td><td>8794</td><td>tags=26%, list=18%, signal=32%</td></tr><tr><td class='lessen'>57</td><td>23.1 NUCLEOTIDE METABOLISM.SYNTHESIS</td><td></td><td>44</td><td>0.30</td><td>0.82</td><td>0.892</td><td>1.000</td><td>1.000</td><td>7603</td><td>tags=41%, list=15%, signal=48%</td></tr><tr><td class='lessen'>58</td><td>29.5.11.20 PROTEIN.DEGRADATION.UBIQUITIN.PROTEASOM</td><td></td><td>73</td><td>0.27</td><td>0.81</td><td>0.952</td><td>1.000</td><td>1.000</td><td>9363</td><td>tags=47%, list=19%, signal=57%</td></tr><tr><td class='lessen'>59</td><td>29.5.11.4.5.2 PROTEIN.DEGRADATION.UBIQUITIN.E3.BTB/POZ CULLIN3.BTB/POZ</td><td></td><td>17</td><td>0.36</td><td>0.81</td><td>0.777</td><td>1.000</td><td>1.000</td><td>6731</td><td>tags=47%, list=14%, signal=54%</td></tr><tr><td class='lessen'>60</td><td>5.10 FERMENTATION.ALDEHYDE DEHYDROGENASE</td><td></td><td>18</td><td>0.35</td><td>0.81</td><td>0.759</td><td>1.000</td><td>1.000</td><td>4269</td><td>tags=11%, list=9%, signal=12%</td></tr><tr><td class='lessen'>61</td><td>29.5.11.4.1 PROTEIN.DEGRADATION.UBIQUITIN.E3.HECT</td><td></td><td>43</td><td>0.29</td><td>0.80</td><td>0.947</td><td>1.000</td><td>1.000</td><td>6665</td><td>tags=12%, list=14%, signal=13%</td></tr><tr><td class='lessen'>62</td><td>29.4.1.55 PROTEIN.POSTRANSLATIONAL MODIFICATION.KINASE.RECEPTOR LIKE CYTOPLASMATIC KINASE V</td><td></td><td>15</td><td>0.38</td><td>0.79</td><td>0.773</td><td>1.000</td><td>1.000</td><td>2284</td><td>tags=20%, list=5%, signal=21%</td></tr><tr><td class='lessen'>63</td><td>21.1 REDOX.THIOREDOXIN</td><td></td><td>79</td><td>0.26</td><td>0.79</td><td>1.000</td><td>1.000</td><td>1.000</td><td>4417</td><td>tags=18%, list=9%, signal=19%</td></tr><tr><td class='lessen'>64</td><td>29.5.9 PROTEIN.DEGRADATION.AAA TYPE</td><td></td><td>59</td><td>0.27</td><td>0.79</td><td>1.000</td><td>1.000</td><td>1.000</td><td>6193</td><td>tags=24%, list=13%, signal=27%</td></tr><tr><td class='lessen'>65</td><td>23.4 NUCLEOTIDE METABOLISM.PHOSPHOTRANSFER AND PYROPHOSPHATASES</td><td></td><td>43</td><td>0.28</td><td>0.78</td><td>0.976</td><td>1.000</td><td>1.000</td><td>3120</td><td>tags=16%, list=6%, signal=17%</td></tr><tr><td class='lessen'>66</td><td>17.1.3 HORMONE METABOLISM.ABSCISIC ACID.INDUCED-REGULATED-RESPONSIVE-ACTIVATED</td><td></td><td>32</td><td>0.31</td><td>0.77</td><td>0.873</td><td>1.000</td><td>1.000</td><td>6450</td><td>tags=41%, list=13%, signal=47%</td></tr><tr><td class='lessen'>67</td><td>9.1.1 MITOCHONDRIAL ELECTRON TRANSPORT / ATP SYNTHESIS.NADH-DH (TYPE I).COMPLEX I</td><td></td><td>21</td><td>0.32</td><td>0.77</td><td>0.880</td><td>1.000</td><td>1.000</td><td>8600</td><td>tags=38%, list=17%, signal=46%</td></tr><tr><td class='lessen'>68</td><td>29.3.4.2 PROTEIN.TARGETING.SECRETORY PATHWAY.GOLGI</td><td></td><td>19</td><td>0.33</td><td>0.76</td><td>0.879</td><td>1.000</td><td>1.000</td><td>5839</td><td>tags=42%, list=12%, signal=48%</td></tr><tr><td class='lessen'>69</td><td>29.2.5 PROTEIN.SYNTHESIS.RELEASE</td><td></td><td>17</td><td>0.34</td><td>0.75</td><td>0.863</td><td>1.000</td><td>1.000</td><td>6790</td><td>tags=41%, list=14%, signal=48%</td></tr><tr><td class='lessen'>70</td><td>27.3.55 RNA.REGULATION OF TRANSCRIPTION.HDA</td><td></td><td>16</td><td>0.34</td><td>0.74</td><td>0.865</td><td>1.000</td><td>1.000</td><td>8064</td><td>tags=50%, list=16%, signal=60%</td></tr><tr><td class='lessen'>71</td><td>17.3.1.2 HORMONE METABOLISM.BRASSINOSTEROID.SYNTHESIS-DEGRADATION.STEROLS</td><td></td><td>20</td><td>0.31</td><td>0.74</td><td>0.914</td><td>1.000</td><td>1.000</td><td>5193</td><td>tags=20%, list=11%, signal=22%</td></tr><tr><td class='lessen'>72</td><td>27.3.40 RNA.REGULATION OF TRANSCRIPTION.AUX/IAA FAMILY</td><td></td><td>26</td><td>0.30</td><td>0.73</td><td>0.966</td><td>1.000</td><td>1.000</td><td>602</td><td>tags=12%, list=1%, signal=12%</td></tr><tr><td class='lessen'>73</td><td>8.1 TCA / ORGANIC TRANSFORMATION.TCA</td><td></td><td>52</td><td>0.24</td><td>0.72</td><td>1.000</td><td>1.000</td><td>1.000</td><td>7998</td><td>tags=37%, list=16%, signal=44%</td></tr><tr><td class='lessen'>74</td><td>29.3.1 PROTEIN.TARGETING.NUCLEUS</td><td></td><td>62</td><td>0.24</td><td>0.71</td><td>1.000</td><td>1.000</td><td>1.000</td><td>6505</td><td>tags=24%, list=13%, signal=28%</td></tr><tr><td class='lessen'>75</td><td>29.3.4.3 PROTEIN.TARGETING.SECRETORY PATHWAY.VACUOLE</td><td></td><td>44</td><td>0.25</td><td>0.71</td><td>1.000</td><td>0.990</td><td>1.000</td><td>7414</td><td>tags=25%, list=15%, signal=29%</td></tr><tr><td class='lessen'>76</td><td>29.5.11.4.5 PROTEIN.DEGRADATION.UBIQUITIN.E3.BTB/POZ CULLIN3</td><td></td><td>20</td><td>0.30</td><td>0.68</td><td>0.976</td><td>0.993</td><td>1.000</td><td>6731</td><td>tags=40%, list=14%, signal=46%</td></tr><tr><td class='lessen'>77</td><td>9.9 MITOCHONDRIAL ELECTRON TRANSPORT / ATP SYNTHESIS.F1-ATPASE</td><td></td><td>63</td><td>0.23</td><td>0.68</td><td>1.000</td><td>0.982</td><td>1.000</td><td>8509</td><td>tags=21%, list=17%, signal=25%</td></tr><tr><td class='lessen'>78</td><td>20.2 STRESS.ABIOTIC</td><td></td><td>496</td><td>0.37</td><td>---</td><td>---</td><td>1.0</td><td>0.0</td><td>6091</td><td>tags=25%, list=12%, signal=29%</td></tr><tr><td class='lessen'>79</td><td>20.2.1 STRESS.ABIOTIC.HEAT</td><td></td><td>372</td><td>0.35495248</td><td>---</td><td>---</td><td>1.0</td><td>0.0</td><td>6347</td><td>tags=27%, list=13%, signal=31%</td></tr><tr><td class='lessen'>80</td><td>27.4 RNA.RNA BINDING</td><td></td><td>297</td><td>0.25911614</td><td>---</td><td>---</td><td>1.0</td><td>0.0</td><td>7637</td><td>tags=30%, list=15%, signal=35%</td></tr><tr><td class='lessen'>81</td><td>29.2.1.2 PROTEIN.SYNTHESIS.RIBOSOMAL PROTEIN.EUKARYOTIC</td><td></td><td>324</td><td>0.22212635</td><td>---</td><td>---</td><td>1.0</td><td>0.0</td><td>9448</td><td>tags=40%, list=19%, signal=50%</td></tr><caption class='table'>Table: Gene sets enriched in phenotype <b>PS218_2h (4 samples)<b>&nbsp<a href='gsea_report_for_PS218_2h_1610970986809.tsv'>[plain text format]</a></caption></table></div><br></body></html>