<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/1999/REC-html401-19991224/loose.dtd">
<html><head><title>Report for PS216_2h 1610970977720 [GSEA]</title><link href='xtools.css' rel='stylesheet'><link href='http://www.gsea-msigdb.org/gsea/images/icon_16x16.png' rel='shortcut icon'></head><body><div class='richTable'><table cols='11' border='1'><th class='richTable'></th><th class='richTable'>GS<br> follow link to MSigDB</th><th class='richTable'>GS DETAILS</th><th class='richTable'>SIZE</th><th class='richTable'>ES</th><th class='richTable'>NES</th><th class='richTable'>NOM p-val</th><th class='richTable'>FDR q-val</th><th class='richTable'>FWER p-val</th><th class='richTable'>RANK AT MAX</th><th class='richTable'>LEADING EDGE</th><tr><td class='lessen'>1</td><td>20.2.4 STRESS.ABIOTIC.TOUCH/WOUNDING</td><td><a href='20.2.4 STRESS.ABIOTIC.TOUCH/WOUNDING.html'>Details ...</a></td><td>27</td><td>0.87</td><td>2.32</td><td>0.000</td><td>0.000</td><td>0.000</td><td>1745</td><td>tags=52%, list=4%, signal=54%</td></tr><tr><td class='lessen'>2</td><td>29.2.1.2.1 PROTEIN.SYNTHESIS.RIBOSOMAL PROTEIN.EUKARYOTIC.40S SUBUNIT</td><td><a href='29.2.1.2.1 PROTEIN.SYNTHESIS.RIBOSOMAL PROTEIN.EUKARYOTIC.40S SUBUNIT.html'>Details ...</a></td><td>123</td><td>0.63</td><td>2.05</td><td>0.000</td><td>0.008</td><td>0.006</td><td>8908</td><td>tags=73%, list=18%, signal=89%</td></tr><tr><td class='lessen'>3</td><td>27.3.36 RNA.REGULATION OF TRANSCRIPTION.ARGONAUTE TRANSCRIPTION FACTOR FAMILY</td><td><a href='27.3.36 RNA.REGULATION OF TRANSCRIPTION.ARGONAUTE TRANSCRIPTION FACTOR FAMILY.html'>Details ...</a></td><td>18</td><td>0.83</td><td>1.96</td><td>0.011</td><td>0.025</td><td>0.025</td><td>4834</td><td>tags=61%, list=10%, signal=68%</td></tr><tr><td class='lessen'>4</td><td>27.3.59 RNA.REGULATION OF TRANSCRIPTION.METHYL BINDING DOMAIN PROTEINS</td><td><a href='27.3.59 RNA.REGULATION OF TRANSCRIPTION.METHYL BINDING DOMAIN PROTEINS.html'>Details ...</a></td><td>26</td><td>0.65</td><td>1.66</td><td>0.016</td><td>0.227</td><td>0.260</td><td>3553</td><td>tags=27%, list=7%, signal=29%</td></tr><tr><td class='lessen'>5</td><td>29.5.11.3 PROTEIN.DEGRADATION.UBIQUITIN.E2</td><td><a href='29.5.11.3 PROTEIN.DEGRADATION.UBIQUITIN.E2.html'>Details ...</a></td><td>58</td><td>0.55</td><td>1.62</td><td>0.000</td><td>0.234</td><td>0.327</td><td>8223</td><td>tags=55%, list=17%, signal=66%</td></tr><tr><td class='lessen'>6</td><td>27.3.16 RNA.REGULATION OF TRANSCRIPTION.CCAAT BOX BINDING FACTOR FAMILY, HAP5</td><td><a href='27.3.16 RNA.REGULATION OF TRANSCRIPTION.CCAAT BOX BINDING FACTOR FAMILY, HAP5.html'>Details ...</a></td><td>17</td><td>0.72</td><td>1.61</td><td>0.024</td><td>0.209</td><td>0.345</td><td>6839</td><td>tags=53%, list=14%, signal=61%</td></tr><tr><td class='lessen'>7</td><td>20.1.7.6.1 STRESS.BIOTIC.PR-PROTEINS.PR6 (PROTEINASE INHIBITORS).TRYPSIN INHIBITOR</td><td><a href='20.1.7.6.1 STRESS.BIOTIC.PR-PROTEINS.PR6 (PROTEINASE INHIBITORS).TRYPSIN INHIBITOR.html'>Details ...</a></td><td>25</td><td>0.63</td><td>1.57</td><td>0.008</td><td>0.224</td><td>0.408</td><td>2187</td><td>tags=40%, list=4%, signal=42%</td></tr><tr><td class='lessen'>8</td><td>29.5.11.20 PROTEIN.DEGRADATION.UBIQUITIN.PROTEASOM</td><td><a href='29.5.11.20 PROTEIN.DEGRADATION.UBIQUITIN.PROTEASOM.html'>Details ...</a></td><td>73</td><td>0.48</td><td>1.51</td><td>0.000</td><td>0.280</td><td>0.534</td><td>8893</td><td>tags=59%, list=18%, signal=72%</td></tr><tr><td class='lessen'>9</td><td>9.1 MITOCHONDRIAL ELECTRON TRANSPORT / ATP SYNTHESIS.NADH-DH (TYPE I)</td><td><a href='9.1 MITOCHONDRIAL ELECTRON TRANSPORT / ATP SYNTHESIS.NADH-DH (TYPE I).html'>Details ...</a></td><td>74</td><td>0.48</td><td>1.48</td><td>0.000</td><td>0.298</td><td>0.590</td><td>8055</td><td>tags=34%, list=16%, signal=40%</td></tr><tr><td class='lessen'>10</td><td>27.3.44 RNA.REGULATION OF TRANSCRIPTION.CHROMATIN REMODELING FACTORS</td><td><a href='27.3.44 RNA.REGULATION OF TRANSCRIPTION.CHROMATIN REMODELING FACTORS.html'>Details ...</a></td><td>63</td><td>0.46</td><td>1.46</td><td>0.000</td><td>0.320</td><td>0.665</td><td>4726</td><td>tags=27%, list=10%, signal=30%</td></tr><tr><td class='lessen'>11</td><td>29.2.1.1.2 PROTEIN.SYNTHESIS.RIBOSOMAL PROTEIN.PROKARYOTIC.MITOCHONDRION</td><td><a href='29.2.1.1.2 PROTEIN.SYNTHESIS.RIBOSOMAL PROTEIN.PROKARYOTIC.MITOCHONDRION.html'>Details ...</a></td><td>26</td><td>0.54</td><td>1.42</td><td>0.017</td><td>0.354</td><td>0.738</td><td>7479</td><td>tags=42%, list=15%, signal=50%</td></tr><tr><td class='lessen'>12</td><td>13.1.4.1 AMINO ACID METABOLISM.SYNTHESIS.BRANCHED CHAIN GROUP.COMMON</td><td><a href='13.1.4.1 AMINO ACID METABOLISM.SYNTHESIS.BRANCHED CHAIN GROUP.COMMON.html'>Details ...</a></td><td>16</td><td>0.61</td><td>1.41</td><td>0.073</td><td>0.341</td><td>0.754</td><td>8036</td><td>tags=75%, list=16%, signal=90%</td></tr><tr><td class='lessen'>13</td><td>23.1 NUCLEOTIDE METABOLISM.SYNTHESIS</td><td><a href='23.1 NUCLEOTIDE METABOLISM.SYNTHESIS.html'>Details ...</a></td><td>44</td><td>0.48</td><td>1.40</td><td>0.041</td><td>0.332</td><td>0.776</td><td>5930</td><td>tags=48%, list=12%, signal=54%</td></tr><tr><td class='lessen'>14</td><td>17.3.1.2 HORMONE METABOLISM.BRASSINOSTEROID.SYNTHESIS-DEGRADATION.STEROLS</td><td><a href='17.3.1.2 HORMONE METABOLISM.BRASSINOSTEROID.SYNTHESIS-DEGRADATION.STEROLS.html'>Details ...</a></td><td>20</td><td>0.58</td><td>1.39</td><td>0.070</td><td>0.328</td><td>0.797</td><td>6731</td><td>tags=40%, list=14%, signal=46%</td></tr><tr><td class='lessen'>15</td><td>28.1.3.2.1 DNA.SYNTHESIS/CHROMATIN STRUCTURE.HISTONE.CORE.H2A</td><td><a href='28.1.3.2.1 DNA.SYNTHESIS/CHROMATIN STRUCTURE.HISTONE.CORE.H2A.html'>Details ...</a></td><td>26</td><td>0.54</td><td>1.38</td><td>0.064</td><td>0.323</td><td>0.814</td><td>8951</td><td>tags=38%, list=18%, signal=47%</td></tr><tr><td class='lessen'>16</td><td>29.3.1 PROTEIN.TARGETING.NUCLEUS</td><td><a href='29.3.1 PROTEIN.TARGETING.NUCLEUS.html'>Details ...</a></td><td>62</td><td>0.45</td><td>1.37</td><td>0.043</td><td>0.329</td><td>0.838</td><td>7408</td><td>tags=53%, list=15%, signal=63%</td></tr><tr><td class='lessen'>17</td><td>27.3.23 RNA.REGULATION OF TRANSCRIPTION.HEAT-SHOCK TRANSCRIPTION FACTOR FAMILY (HSF)</td><td><a href='27.3.23 RNA.REGULATION OF TRANSCRIPTION.HEAT-SHOCK TRANSCRIPTION FACTOR FAMILY (HSF).html'>Details ...</a></td><td>33</td><td>0.50</td><td>1.36</td><td>0.055</td><td>0.324</td><td>0.849</td><td>5493</td><td>tags=36%, list=11%, signal=41%</td></tr><tr><td class='lessen'>18</td><td>28.1.3 DNA.SYNTHESIS/CHROMATIN STRUCTURE.HISTONE</td><td><a href='28.1.3 DNA.SYNTHESIS/CHROMATIN STRUCTURE.HISTONE.html'>Details ...</a></td><td>83</td><td>0.43</td><td>1.35</td><td>0.000</td><td>0.317</td><td>0.857</td><td>8951</td><td>tags=46%, list=18%, signal=56%</td></tr><tr><td class='lessen'>19</td><td>27.1.3 RNA.PROCESSING.3' END PROCESSING</td><td><a href="27.1.3 RNA.PROCESSING.3' END PROCESSING.html">Details ...</a></td><td>41</td><td>0.50</td><td>1.35</td><td>0.071</td><td>0.309</td><td>0.862</td><td>5434</td><td>tags=49%, list=11%, signal=55%</td></tr><tr><td class='lessen'>20</td><td>27.3.14 RNA.REGULATION OF TRANSCRIPTION.CCAAT BOX BINDING FACTOR FAMILY, HAP2</td><td><a href='27.3.14 RNA.REGULATION OF TRANSCRIPTION.CCAAT BOX BINDING FACTOR FAMILY, HAP2.html'>Details ...</a></td><td>15</td><td>0.60</td><td>1.35</td><td>0.095</td><td>0.294</td><td>0.862</td><td>4864</td><td>tags=47%, list=10%, signal=52%</td></tr><tr><td class='lessen'>21</td><td>29.5.11.4.3.1 PROTEIN.DEGRADATION.UBIQUITIN.E3.SCF.SKP</td><td><a href='29.5.11.4.3.1 PROTEIN.DEGRADATION.UBIQUITIN.E3.SCF.SKP.html'>Details ...</a></td><td>56</td><td>0.44</td><td>1.35</td><td>0.037</td><td>0.283</td><td>0.863</td><td>4399</td><td>tags=7%, list=9%, signal=8%</td></tr><tr><td class='lessen'>22</td><td>9.1.2 MITOCHONDRIAL ELECTRON TRANSPORT / ATP SYNTHESIS.NADH-DH (TYPE I).LOCALISATION NOT CLEAR</td><td><a href='9.1.2 MITOCHONDRIAL ELECTRON TRANSPORT / ATP SYNTHESIS.NADH-DH (TYPE I).LOCALISATION NOT CLEAR.html'>Details ...</a></td><td>53</td><td>0.44</td><td>1.35</td><td>0.045</td><td>0.273</td><td>0.864</td><td>8055</td><td>tags=28%, list=16%, signal=34%</td></tr><tr><td class='lessen'>23</td><td>28.1.3.2 DNA.SYNTHESIS/CHROMATIN STRUCTURE.HISTONE.CORE</td><td><a href='28.1.3.2 DNA.SYNTHESIS/CHROMATIN STRUCTURE.HISTONE.CORE.html'>Details ...</a></td><td>76</td><td>0.43</td><td>1.32</td><td>0.045</td><td>0.306</td><td>0.906</td><td>8951</td><td>tags=47%, list=18%, signal=58%</td></tr><tr><td class='lessen'>24</td><td>9.1.1 MITOCHONDRIAL ELECTRON TRANSPORT / ATP SYNTHESIS.NADH-DH (TYPE I).COMPLEX I</td><td><a href='9.1.1 MITOCHONDRIAL ELECTRON TRANSPORT / ATP SYNTHESIS.NADH-DH (TYPE I).COMPLEX I.html'>Details ...</a></td><td>21</td><td>0.54</td><td>1.31</td><td>0.081</td><td>0.303</td><td>0.911</td><td>8040</td><td>tags=48%, list=16%, signal=57%</td></tr><tr><td class='lessen'>25</td><td>13.1.1 AMINO ACID METABOLISM.SYNTHESIS.CENTRAL AMINO ACID METABOLISM</td><td><a href='13.1.1 AMINO ACID METABOLISM.SYNTHESIS.CENTRAL AMINO ACID METABOLISM.html'>Details ...</a></td><td>24</td><td>0.53</td><td>1.29</td><td>0.098</td><td>0.318</td><td>0.934</td><td>5208</td><td>tags=54%, list=11%, signal=61%</td></tr><tr><td class='lessen'>26</td><td>13.1.5 AMINO ACID METABOLISM.SYNTHESIS.SERINE-GLYCINE-CYSTEINE GROUP</td><td><a href='13.1.5 AMINO ACID METABOLISM.SYNTHESIS.SERINE-GLYCINE-CYSTEINE GROUP.html'>Details ...</a></td><td>39</td><td>0.45</td><td>1.29</td><td>0.038</td><td>0.309</td><td>0.938</td><td>4352</td><td>tags=26%, list=9%, signal=28%</td></tr><tr><td class='lessen'>27</td><td>29.5.11.4.1 PROTEIN.DEGRADATION.UBIQUITIN.E3.HECT</td><td><a href='29.5.11.4.1 PROTEIN.DEGRADATION.UBIQUITIN.E3.HECT.html'>Details ...</a></td><td>43</td><td>0.44</td><td>1.26</td><td>0.042</td><td>0.344</td><td>0.962</td><td>2129</td><td>tags=9%, list=4%, signal=10%</td></tr><tr><td class='lessen'>28</td><td>11.8.1 LIPID METABOLISM.EXOTICS (STEROIDS, SQUALENE ETC).SPHINGOLIPIDS</td><td><a href='11.8.1 LIPID METABOLISM.EXOTICS (STEROIDS, SQUALENE ETC).SPHINGOLIPIDS.html'>Details ...</a></td><td>43</td><td>0.44</td><td>1.26</td><td>0.093</td><td>0.332</td><td>0.962</td><td>4882</td><td>tags=23%, list=10%, signal=26%</td></tr><tr><td class='lessen'>29</td><td>26.16 MISC.MYROSINASES-LECTIN-JACALIN</td><td><a href='26.16 MISC.MYROSINASES-LECTIN-JACALIN.html'>Details ...</a></td><td>33</td><td>0.46</td><td>1.24</td><td>0.111</td><td>0.365</td><td>0.974</td><td>1143</td><td>tags=15%, list=2%, signal=16%</td></tr><tr><td class='lessen'>30</td><td>11.1.15 LIPID METABOLISM.FA SYNTHESIS AND FA ELONGATION.ACP DESATURASE</td><td><a href='11.1.15 LIPID METABOLISM.FA SYNTHESIS AND FA ELONGATION.ACP DESATURASE.html'>Details ...</a></td><td>17</td><td>0.54</td><td>1.23</td><td>0.155</td><td>0.371</td><td>0.981</td><td>1759</td><td>tags=12%, list=4%, signal=12%</td></tr><tr><td class='lessen'>31</td><td>18.4 CO-FACTOR AND VITAMINE METABOLISM.PANTOTHENATE</td><td><a href='18.4 CO-FACTOR AND VITAMINE METABOLISM.PANTOTHENATE.html'>Details ...</a></td><td>16</td><td>0.54</td><td>1.22</td><td>0.198</td><td>0.367</td><td>0.982</td><td>7787</td><td>tags=56%, list=16%, signal=67%</td></tr><tr><td class='lessen'>32</td><td>30.11.1 SIGNALLING.LIGHT.COP9 SIGNALOSOME</td><td><a href='30.11.1 SIGNALLING.LIGHT.COP9 SIGNALOSOME.html'>Details ...</a></td><td>22</td><td>0.48</td><td>1.19</td><td>0.158</td><td>0.422</td><td>0.990</td><td>5946</td><td>tags=45%, list=12%, signal=52%</td></tr><tr><td class='lessen'>33</td><td>29.2.1.2.2 PROTEIN.SYNTHESIS.RIBOSOMAL PROTEIN.EUKARYOTIC.60S SUBUNIT</td><td><a href='29.2.1.2.2 PROTEIN.SYNTHESIS.RIBOSOMAL PROTEIN.EUKARYOTIC.60S SUBUNIT.html'>Details ...</a></td><td>200</td><td>0.36</td><td>1.18</td><td>0.000</td><td>0.426</td><td>0.993</td><td>9251</td><td>tags=61%, list=19%, signal=74%</td></tr><tr><td class='lessen'>34</td><td>29.2.4 PROTEIN.SYNTHESIS.ELONGATION</td><td><a href='29.2.4 PROTEIN.SYNTHESIS.ELONGATION.html'>Details ...</a></td><td>41</td><td>0.42</td><td>1.18</td><td>0.069</td><td>0.429</td><td>0.995</td><td>8299</td><td>tags=51%, list=17%, signal=62%</td></tr><tr><td class='lessen'>35</td><td>20.1.7.6 STRESS.BIOTIC.PR-PROTEINS.PR6 (PROTEINASE INHIBITORS)</td><td><a href='20.1.7.6 STRESS.BIOTIC.PR-PROTEINS.PR6 (PROTEINASE INHIBITORS).html'>Details ...</a></td><td>55</td><td>0.39</td><td>1.17</td><td>0.184</td><td>0.419</td><td>0.995</td><td>2187</td><td>tags=20%, list=4%, signal=21%</td></tr><tr><td class='lessen'>36</td><td>13.1.4 AMINO ACID METABOLISM.SYNTHESIS.BRANCHED CHAIN GROUP</td><td><a href='13.1.4 AMINO ACID METABOLISM.SYNTHESIS.BRANCHED CHAIN GROUP.html'>Details ...</a></td><td>24</td><td>0.44</td><td>1.15</td><td>0.155</td><td>0.459</td><td>0.998</td><td>5887</td><td>tags=46%, list=12%, signal=52%</td></tr><tr><td class='lessen'>37</td><td>9.9 MITOCHONDRIAL ELECTRON TRANSPORT / ATP SYNTHESIS.F1-ATPASE</td><td><a href='9.9 MITOCHONDRIAL ELECTRON TRANSPORT / ATP SYNTHESIS.F1-ATPASE.html'>Details ...</a></td><td>63</td><td>0.38</td><td>1.15</td><td>0.150</td><td>0.458</td><td>0.999</td><td>9512</td><td>tags=33%, list=19%, signal=41%</td></tr><tr><td class='lessen'>38</td><td>13.1.5.3 AMINO ACID METABOLISM.SYNTHESIS.SERINE-GLYCINE-CYSTEINE GROUP.CYSTEINE</td><td><a href='13.1.5.3 AMINO ACID METABOLISM.SYNTHESIS.SERINE-GLYCINE-CYSTEINE GROUP.CYSTEINE.html'>Details ...</a></td><td>32</td><td>0.42</td><td>1.14</td><td>0.198</td><td>0.463</td><td>0.999</td><td>4352</td><td>tags=25%, list=9%, signal=27%</td></tr><tr><td class='lessen'>39</td><td>29.2.2.3.4 PROTEIN.SYNTHESIS.RIBOSOME BIOGENESIS.PRE-RRNA PROCESSING AND MODIFICATIONS.WD-REPEAT PROTEINS</td><td><a href='29.2.2.3.4 PROTEIN.SYNTHESIS.RIBOSOME BIOGENESIS.PRE-RRNA PROCESSING AND MODIFICATIONS.WD-REPEAT PROTEINS.html'>Details ...</a></td><td>35</td><td>0.41</td><td>1.12</td><td>0.202</td><td>0.508</td><td>1.000</td><td>7005</td><td>tags=29%, list=14%, signal=33%</td></tr><tr><td class='lessen'>40</td><td>26.24 MISC.GCN5-RELATED N-ACETYLTRANSFERASE</td><td><a href='26.24 MISC.GCN5-RELATED N-ACETYLTRANSFERASE.html'>Details ...</a></td><td>58</td><td>0.36</td><td>1.11</td><td>0.212</td><td>0.512</td><td>1.000</td><td>5642</td><td>tags=38%, list=11%, signal=43%</td></tr><tr><td class='lessen'>41</td><td>27.3.30 RNA.REGULATION OF TRANSCRIPTION.TRIPLE-HELIX TRANSCRIPTION FACTOR FAMILY (TRIHELIX)</td><td><a href='27.3.30 RNA.REGULATION OF TRANSCRIPTION.TRIPLE-HELIX TRANSCRIPTION FACTOR FAMILY (TRIHELIX).html'>Details ...</a></td><td>32</td><td>0.40</td><td>1.11</td><td>0.250</td><td>0.509</td><td>1.000</td><td>2585</td><td>tags=22%, list=5%, signal=23%</td></tr><tr><td class='lessen'>42</td><td>29.2.1.1.2.1 PROTEIN.SYNTHESIS.RIBOSOMAL PROTEIN.PROKARYOTIC.MITOCHONDRION.30S SUBUNIT</td><td><a href='29.2.1.1.2.1 PROTEIN.SYNTHESIS.RIBOSOMAL PROTEIN.PROKARYOTIC.MITOCHONDRION.30S SUBUNIT.html'>Details ...</a></td><td>22</td><td>0.45</td><td>1.10</td><td>0.240</td><td>0.503</td><td>1.000</td><td>7479</td><td>tags=32%, list=15%, signal=37%</td></tr><tr><td class='lessen'>43</td><td>9.7 MITOCHONDRIAL ELECTRON TRANSPORT / ATP SYNTHESIS.CYTOCHROME C OXIDASE</td><td><a href='9.7 MITOCHONDRIAL ELECTRON TRANSPORT / ATP SYNTHESIS.CYTOCHROME C OXIDASE.html'>Details ...</a></td><td>41</td><td>0.39</td><td>1.10</td><td>0.200</td><td>0.509</td><td>1.000</td><td>1648</td><td>tags=7%, list=3%, signal=8%</td></tr><tr><td class='lessen'>44</td><td>27.1.1 RNA.PROCESSING.SPLICING</td><td><a href='27.1.1 RNA.PROCESSING.SPLICING.html'>Details ...</a></td><td>88</td><td>0.35</td><td>1.09</td><td>0.235</td><td>0.511</td><td>1.000</td><td>8973</td><td>tags=52%, list=18%, signal=64%</td></tr><tr><td class='lessen'>45</td><td>27.1.2 RNA.PROCESSING.RNA HELICASE</td><td><a href='27.1.2 RNA.PROCESSING.RNA HELICASE.html'>Details ...</a></td><td>57</td><td>0.36</td><td>1.08</td><td>0.216</td><td>0.532</td><td>1.000</td><td>7960</td><td>tags=46%, list=16%, signal=54%</td></tr><tr><td class='lessen'>46</td><td>29.3.4.2 PROTEIN.TARGETING.SECRETORY PATHWAY.GOLGI</td><td><a href='29.3.4.2 PROTEIN.TARGETING.SECRETORY PATHWAY.GOLGI.html'>Details ...</a></td><td>19</td><td>0.45</td><td>1.07</td><td>0.292</td><td>0.530</td><td>1.000</td><td>5408</td><td>tags=37%, list=11%, signal=41%</td></tr><tr><td class='lessen'>47</td><td>4.1 GLYCOLYSIS.CYTOSOLIC BRANCH</td><td><a href='4.1 GLYCOLYSIS.CYTOSOLIC BRANCH.html'>Details ...</a></td><td>63</td><td>0.35</td><td>1.07</td><td>0.214</td><td>0.537</td><td>1.000</td><td>7983</td><td>tags=46%, list=16%, signal=55%</td></tr><tr><td class='lessen'>48</td><td>28.2 DNA.REPAIR</td><td><a href='28.2 DNA.REPAIR.html'>Details ...</a></td><td>138</td><td>0.31</td><td>1.05</td><td>0.200</td><td>0.588</td><td>1.000</td><td>5205</td><td>tags=24%, list=11%, signal=27%</td></tr><tr><td class='lessen'>49</td><td>29.5.11.1 PROTEIN.DEGRADATION.UBIQUITIN.UBIQUITIN</td><td><a href='29.5.11.1 PROTEIN.DEGRADATION.UBIQUITIN.UBIQUITIN.html'>Details ...</a></td><td>54</td><td>0.36</td><td>1.04</td><td>0.296</td><td>0.576</td><td>1.000</td><td>9414</td><td>tags=50%, list=19%, signal=62%</td></tr><tr><td class='lessen'>50</td><td>27.3.57 RNA.REGULATION OF TRANSCRIPTION.JUMONJI FAMILY</td><td><a href='27.3.57 RNA.REGULATION OF TRANSCRIPTION.JUMONJI FAMILY.html'>Details ...</a></td><td>32</td><td>0.38</td><td>1.04</td><td>0.365</td><td>0.589</td><td>1.000</td><td>4659</td><td>tags=22%, list=9%, signal=24%</td></tr><tr><td class='lessen'>51</td><td>29.5.2 PROTEIN.DEGRADATION.AUTOPHAGY</td><td></td><td>22</td><td>0.42</td><td>1.03</td><td>0.380</td><td>0.599</td><td>1.000</td><td>5228</td><td>tags=41%, list=11%, signal=46%</td></tr><tr><td class='lessen'>52</td><td>5 FERMENTATION</td><td></td><td>44</td><td>0.35</td><td>1.02</td><td>0.403</td><td>0.626</td><td>1.000</td><td>931</td><td>tags=14%, list=2%, signal=14%</td></tr><tr><td class='lessen'>53</td><td>29.3.4.3 PROTEIN.TARGETING.SECRETORY PATHWAY.VACUOLE</td><td></td><td>44</td><td>0.35</td><td>1.02</td><td>0.414</td><td>0.616</td><td>1.000</td><td>8331</td><td>tags=27%, list=17%, signal=33%</td></tr><tr><td class='lessen'>54</td><td>23.5 NUCLEOTIDE METABOLISM.DEOXYNUCLEOTIDE METABOLISM</td><td></td><td>17</td><td>0.44</td><td>1.01</td><td>0.392</td><td>0.620</td><td>1.000</td><td>6404</td><td>tags=47%, list=13%, signal=54%</td></tr><tr><td class='lessen'>55</td><td>23.3 NUCLEOTIDE METABOLISM.SALVAGE</td><td></td><td>60</td><td>0.33</td><td>1.00</td><td>0.463</td><td>0.641</td><td>1.000</td><td>5671</td><td>tags=25%, list=11%, signal=28%</td></tr><tr><td class='lessen'>56</td><td>29.3.2 PROTEIN.TARGETING.MITOCHONDRIA</td><td></td><td>39</td><td>0.37</td><td>1.00</td><td>0.385</td><td>0.640</td><td>1.000</td><td>7101</td><td>tags=41%, list=14%, signal=48%</td></tr><tr><td class='lessen'>57</td><td>26.17 MISC.DYNAMIN</td><td></td><td>29</td><td>0.37</td><td>0.99</td><td>0.532</td><td>0.652</td><td>1.000</td><td>6836</td><td>tags=38%, list=14%, signal=44%</td></tr><tr><td class='lessen'>58</td><td>15.2 METAL HANDLING.BINDING, CHELATION AND STORAGE</td><td></td><td>117</td><td>0.29</td><td>0.98</td><td>0.545</td><td>0.686</td><td>1.000</td><td>5206</td><td>tags=18%, list=11%, signal=20%</td></tr><tr><td class='lessen'>59</td><td>27.1.19 RNA.PROCESSING.RIBONUCLEASES</td><td></td><td>95</td><td>0.29</td><td>0.95</td><td>0.810</td><td>0.767</td><td>1.000</td><td>5004</td><td>tags=24%, list=10%, signal=27%</td></tr><tr><td class='lessen'>60</td><td>4 GLYCOLYSIS</td><td></td><td>92</td><td>0.29</td><td>0.95</td><td>0.684</td><td>0.754</td><td>1.000</td><td>7983</td><td>tags=39%, list=16%, signal=47%</td></tr><tr><td class='lessen'>61</td><td>11.8 LIPID METABOLISM.EXOTICS (STEROIDS, SQUALENE ETC)</td><td></td><td>95</td><td>0.30</td><td>0.94</td><td>0.625</td><td>0.776</td><td>1.000</td><td>4978</td><td>tags=23%, list=10%, signal=26%</td></tr><tr><td class='lessen'>62</td><td>29.2.5 PROTEIN.SYNTHESIS.RELEASE</td><td></td><td>17</td><td>0.40</td><td>0.93</td><td>0.515</td><td>0.797</td><td>1.000</td><td>6750</td><td>tags=29%, list=14%, signal=34%</td></tr><tr><td class='lessen'>63</td><td>27.2 RNA.TRANSCRIPTION</td><td></td><td>132</td><td>0.28</td><td>0.92</td><td>0.600</td><td>0.794</td><td>1.000</td><td>6168</td><td>tags=26%, list=13%, signal=29%</td></tr><tr><td class='lessen'>64</td><td>4.3 GLYCOLYSIS.UNCLEAR/DUALLY TARGETED</td><td></td><td>16</td><td>0.40</td><td>0.91</td><td>0.549</td><td>0.826</td><td>1.000</td><td>5110</td><td>tags=25%, list=10%, signal=28%</td></tr><tr><td class='lessen'>65</td><td>8.1 TCA / ORGANIC TRANSFORMATION.TCA</td><td></td><td>52</td><td>0.29</td><td>0.89</td><td>0.786</td><td>0.891</td><td>1.000</td><td>9335</td><td>tags=48%, list=19%, signal=59%</td></tr><tr><td class='lessen'>66</td><td>29.1 PROTEIN.AA ACTIVATION</td><td></td><td>89</td><td>0.28</td><td>0.87</td><td>0.905</td><td>0.936</td><td>1.000</td><td>8086</td><td>tags=39%, list=16%, signal=47%</td></tr><tr><td class='lessen'>67</td><td>29.1.30 PROTEIN.AA ACTIVATION.PSEUDOURIDYLATE SYNTHASE</td><td></td><td>19</td><td>0.37</td><td>0.86</td><td>0.691</td><td>0.929</td><td>1.000</td><td>2809</td><td>tags=37%, list=6%, signal=39%</td></tr><tr><td class='lessen'>68</td><td>28.1.3.2.3 DNA.SYNTHESIS/CHROMATIN STRUCTURE.HISTONE.CORE.H3</td><td></td><td>20</td><td>0.36</td><td>0.82</td><td>0.711</td><td>1.000</td><td>1.000</td><td>8638</td><td>tags=45%, list=18%, signal=55%</td></tr><tr><td class='lessen'>69</td><td>17.1.3 HORMONE METABOLISM.ABSCISIC ACID.INDUCED-REGULATED-RESPONSIVE-ACTIVATED</td><td></td><td>32</td><td>0.31</td><td>0.82</td><td>0.872</td><td>1.000</td><td>1.000</td><td>4017</td><td>tags=22%, list=8%, signal=24%</td></tr><tr><td class='lessen'>70</td><td>17.3.1 HORMONE METABOLISM.BRASSINOSTEROID.SYNTHESIS-DEGRADATION</td><td></td><td>28</td><td>0.31</td><td>0.80</td><td>0.852</td><td>1.000</td><td>1.000</td><td>5396</td><td>tags=32%, list=11%, signal=36%</td></tr><tr><td class='lessen'>71</td><td>7.1 OPP.OXIDATIVE PP</td><td></td><td>15</td><td>0.35</td><td>0.80</td><td>0.763</td><td>1.000</td><td>1.000</td><td>8877</td><td>tags=60%, list=18%, signal=73%</td></tr><tr><td class='lessen'>72</td><td>21.2.2 REDOX.ASCORBATE AND GLUTATHIONE.GLUTATHIONE</td><td></td><td>15</td><td>0.34</td><td>0.77</td><td>0.850</td><td>1.000</td><td>1.000</td><td>9204</td><td>tags=67%, list=19%, signal=82%</td></tr><tr><td class='lessen'>73</td><td>5.10 FERMENTATION.ALDEHYDE DEHYDROGENASE</td><td></td><td>18</td><td>0.33</td><td>0.75</td><td>0.890</td><td>1.000</td><td>1.000</td><td>8120</td><td>tags=33%, list=16%, signal=40%</td></tr><tr><td class='lessen'>74</td><td>29.2.1.1.3.2 PROTEIN.SYNTHESIS.RIBOSOMAL PROTEIN.PROKARYOTIC.UNKNOWN ORGANELLAR.50S SUBUNIT</td><td></td><td>30</td><td>0.29</td><td>0.74</td><td>0.947</td><td>1.000</td><td>1.000</td><td>5484</td><td>tags=30%, list=11%, signal=34%</td></tr><tr><td class='lessen'>75</td><td>29.3.3 PROTEIN.TARGETING.CHLOROPLAST</td><td></td><td>37</td><td>0.27</td><td>0.73</td><td>1.000</td><td>1.000</td><td>1.000</td><td>6749</td><td>tags=32%, list=14%, signal=38%</td></tr><tr><td class='lessen'>76</td><td>29.5.11.4.5.2 PROTEIN.DEGRADATION.UBIQUITIN.E3.BTB/POZ CULLIN3.BTB/POZ</td><td></td><td>17</td><td>0.32</td><td>0.73</td><td>0.921</td><td>1.000</td><td>1.000</td><td>5988</td><td>tags=41%, list=12%, signal=47%</td></tr><tr><td class='lessen'>77</td><td>34.1.1 TRANSPORT.P- AND V-ATPASES.H+-TRANSPORTING TWO-SECTOR ATPASE</td><td></td><td>20</td><td>0.30</td><td>0.71</td><td>0.958</td><td>1.000</td><td>1.000</td><td>536</td><td>tags=5%, list=1%, signal=5%</td></tr><tr><td class='lessen'>78</td><td>27.3.62 RNA.REGULATION OF TRANSCRIPTION.NUCLEOSOME/CHROMATIN ASSEMBLY FACTOR GROUP</td><td></td><td>16</td><td>0.31</td><td>0.71</td><td>0.954</td><td>1.000</td><td>1.000</td><td>6477</td><td>tags=38%, list=13%, signal=43%</td></tr><tr><td class='lessen'>79</td><td>27.3.52 RNA.REGULATION OF TRANSCRIPTION.GLOBAL TRANSCRIPTION FACTOR GROUP</td><td></td><td>22</td><td>0.29</td><td>0.70</td><td>0.952</td><td>1.000</td><td>1.000</td><td>8510</td><td>tags=45%, list=17%, signal=55%</td></tr><tr><td class='lessen'>80</td><td>18.7 CO-FACTOR AND VITAMINE METABOLISM.IRON-SULPHUR CLUSTERS</td><td></td><td>16</td><td>0.31</td><td>0.70</td><td>0.957</td><td>1.000</td><td>1.000</td><td>1790</td><td>tags=19%, list=4%, signal=19%</td></tr><tr><td class='lessen'>81</td><td>27.3.49 RNA.REGULATION OF TRANSCRIPTION.GEBP LIKE</td><td></td><td>15</td><td>0.31</td><td>0.70</td><td>0.929</td><td>1.000</td><td>1.000</td><td>4077</td><td>tags=20%, list=8%, signal=22%</td></tr><tr><td class='lessen'>82</td><td>3.2.3 MINOR CHO METABOLISM.TREHALOSE.POTENTIAL TPS/TPP</td><td></td><td>15</td><td>0.32</td><td>0.69</td><td>0.927</td><td>1.000</td><td>1.000</td><td>6120</td><td>tags=53%, list=12%, signal=61%</td></tr><tr><td class='lessen'>83</td><td>27.3.55 RNA.REGULATION OF TRANSCRIPTION.HDA</td><td></td><td>16</td><td>0.30</td><td>0.69</td><td>0.960</td><td>1.000</td><td>1.000</td><td>6344</td><td>tags=38%, list=13%, signal=43%</td></tr><tr><td class='lessen'>84</td><td>13.2.4 AMINO ACID METABOLISM.DEGRADATION.BRANCHED CHAIN GROUP</td><td></td><td>16</td><td>0.30</td><td>0.68</td><td>0.948</td><td>0.997</td><td>1.000</td><td>7764</td><td>tags=38%, list=16%, signal=44%</td></tr><tr><td class='lessen'>85</td><td>29.5.11.4.3.3 PROTEIN.DEGRADATION.UBIQUITIN.E3.SCF.CULLIN</td><td></td><td>36</td><td>0.24</td><td>0.66</td><td>1.000</td><td>0.989</td><td>1.000</td><td>7858</td><td>tags=22%, list=16%, signal=26%</td></tr><tr><td class='lessen'>86</td><td>13.1 AMINO ACID METABOLISM.SYNTHESIS</td><td></td><td>232</td><td>0.26</td><td>---</td><td>---</td><td>1.0</td><td>0.0</td><td>6209</td><td>tags=25%, list=13%, signal=29%</td></tr><tr><td class='lessen'>87</td><td>20.2 STRESS.ABIOTIC</td><td></td><td>496</td><td>0.36976674</td><td>---</td><td>---</td><td>1.0</td><td>0.0</td><td>6246</td><td>tags=29%, list=13%, signal=33%</td></tr><tr><td class='lessen'>88</td><td>20.2.1 STRESS.ABIOTIC.HEAT</td><td></td><td>372</td><td>0.37178904</td><td>---</td><td>---</td><td>1.0</td><td>0.0</td><td>6311</td><td>tags=30%, list=13%, signal=35%</td></tr><tr><td class='lessen'>89</td><td>23 NUCLEOTIDE METABOLISM</td><td></td><td>216</td><td>0.25376242</td><td>---</td><td>---</td><td>1.0</td><td>0.0</td><td>6717</td><td>tags=30%, list=14%, signal=34%</td></tr><tr><td class='lessen'>90</td><td>27.1 RNA.PROCESSING</td><td></td><td>425</td><td>0.31465593</td><td>---</td><td>---</td><td>1.0</td><td>0.0</td><td>7109</td><td>tags=38%, list=14%, signal=44%</td></tr><tr><td class='lessen'>91</td><td>27.4 RNA.RNA BINDING</td><td></td><td>297</td><td>0.32796553</td><td>---</td><td>---</td><td>1.0</td><td>0.0</td><td>8179</td><td>tags=42%, list=17%, signal=50%</td></tr><tr><td class='lessen'>92</td><td>29.2.1.2 PROTEIN.SYNTHESIS.RIBOSOMAL PROTEIN.EUKARYOTIC</td><td></td><td>324</td><td>0.46392548</td><td>---</td><td>---</td><td>1.0</td><td>0.0</td><td>9251</td><td>tags=66%, list=19%, signal=80%</td></tr><tr><td class='lessen'>93</td><td>29.3 PROTEIN.TARGETING</td><td></td><td>367</td><td>0.23594624</td><td>---</td><td>---</td><td>1.0</td><td>0.0</td><td>7434</td><td>tags=31%, list=15%, signal=36%</td></tr><tr><td class='lessen'>94</td><td>9 MITOCHONDRIAL ELECTRON TRANSPORT / ATP SYNTHESIS</td><td></td><td>237</td><td>0.33941048</td><td>---</td><td>---</td><td>1.0</td><td>0.0</td><td>8168</td><td>tags=25%, list=17%, signal=30%</td></tr><caption class='table'>Table: Gene sets enriched in phenotype <b>PS216_2h (4 samples)<b>&nbsp<a href='gsea_report_for_PS216_2h_1610970977720.tsv'>[plain text format]</a></caption></table></div><br></body></html>