<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/1999/REC-html401-19991224/loose.dtd">
<html><head><title>Report for PS218_26h 1610971023665 [GSEA]</title><link href='xtools.css' rel='stylesheet'><link href='http://www.gsea-msigdb.org/gsea/images/icon_16x16.png' rel='shortcut icon'></head><body><div class='richTable'><table cols='11' border='1'><th class='richTable'></th><th class='richTable'>GS<br> follow link to MSigDB</th><th class='richTable'>GS DETAILS</th><th class='richTable'>SIZE</th><th class='richTable'>ES</th><th class='richTable'>NES</th><th class='richTable'>NOM p-val</th><th class='richTable'>FDR q-val</th><th class='richTable'>FWER p-val</th><th class='richTable'>RANK AT MAX</th><th class='richTable'>LEADING EDGE</th><tr><td class='lessen'>1</td><td>29.5.11.4.1 PROTEIN.DEGRADATION.UBIQUITIN.E3.HECT</td><td><a href='29.5.11.4.1 PROTEIN.DEGRADATION.UBIQUITIN.E3.HECT.html'>Details ...</a></td><td>43</td><td>0.77</td><td>1.96</td><td>0.000</td><td>0.003</td><td>0.003</td><td>7642</td><td>tags=33%, list=15%, signal=38%</td></tr><tr><td class='lessen'>2</td><td>27.3.32 RNA.REGULATION OF TRANSCRIPTION.WRKY DOMAIN TRANSCRIPTION FACTOR FAMILY</td><td><a href='27.3.32 RNA.REGULATION OF TRANSCRIPTION.WRKY DOMAIN TRANSCRIPTION FACTOR FAMILY.html'>Details ...</a></td><td>97</td><td>0.65</td><td>1.93</td><td>0.000</td><td>0.004</td><td>0.007</td><td>8062</td><td>tags=45%, list=16%, signal=54%</td></tr><tr><td class='lessen'>3</td><td>20.2.4 STRESS.ABIOTIC.TOUCH/WOUNDING</td><td><a href='20.2.4 STRESS.ABIOTIC.TOUCH/WOUNDING.html'>Details ...</a></td><td>27</td><td>0.81</td><td>1.90</td><td>0.003</td><td>0.004</td><td>0.011</td><td>6483</td><td>tags=78%, list=13%, signal=89%</td></tr><tr><td class='lessen'>4</td><td>16.8.1 SECONDARY METABOLISM.FLAVONOIDS.ANTHOCYANINS</td><td><a href='16.8.1 SECONDARY METABOLISM.FLAVONOIDS.ANTHOCYANINS.html'>Details ...</a></td><td>79</td><td>0.64</td><td>1.82</td><td>0.000</td><td>0.012</td><td>0.042</td><td>4143</td><td>tags=32%, list=8%, signal=34%</td></tr><tr><td class='lessen'>5</td><td>16.8.1.12 SECONDARY METABOLISM.FLAVONOIDS.ANTHOCYANINS.ANTHOCYANIDIN 3-O-GLUCOSYLTRANSFERASE</td><td><a href='16.8.1.12 SECONDARY METABOLISM.FLAVONOIDS.ANTHOCYANINS.ANTHOCYANIDIN 3-O-GLUCOSYLTRANSFERASE.html'>Details ...</a></td><td>62</td><td>0.66</td><td>1.79</td><td>0.000</td><td>0.017</td><td>0.072</td><td>4143</td><td>tags=32%, list=8%, signal=35%</td></tr><tr><td class='lessen'>6</td><td>13.2.7 AMINO ACID METABOLISM.DEGRADATION.HISTIDINE</td><td><a href='13.2.7 AMINO ACID METABOLISM.DEGRADATION.HISTIDINE.html'>Details ...</a></td><td>33</td><td>0.71</td><td>1.75</td><td>0.000</td><td>0.029</td><td>0.143</td><td>10708</td><td>tags=36%, list=22%, signal=46%</td></tr><tr><td class='lessen'>7</td><td>29.5.11.4.5 PROTEIN.DEGRADATION.UBIQUITIN.E3.BTB/POZ CULLIN3</td><td><a href='29.5.11.4.5 PROTEIN.DEGRADATION.UBIQUITIN.E3.BTB/POZ CULLIN3.html'>Details ...</a></td><td>20</td><td>0.75</td><td>1.65</td><td>0.012</td><td>0.096</td><td>0.448</td><td>5822</td><td>tags=55%, list=12%, signal=62%</td></tr><tr><td class='lessen'>8</td><td>20.1.7.6.1 STRESS.BIOTIC.PR-PROTEINS.PR6 (PROTEINASE INHIBITORS).TRYPSIN INHIBITOR</td><td><a href='20.1.7.6.1 STRESS.BIOTIC.PR-PROTEINS.PR6 (PROTEINASE INHIBITORS).TRYPSIN INHIBITOR.html'>Details ...</a></td><td>25</td><td>0.71</td><td>1.64</td><td>0.007</td><td>0.093</td><td>0.484</td><td>5223</td><td>tags=48%, list=11%, signal=54%</td></tr><tr><td class='lessen'>9</td><td>33.30 DEVELOPMENT.MULTITARGET</td><td><a href='33.30 DEVELOPMENT.MULTITARGET.html'>Details ...</a></td><td>20</td><td>0.75</td><td>1.64</td><td>0.012</td><td>0.084</td><td>0.487</td><td>11938</td><td>tags=35%, list=24%, signal=46%</td></tr><tr><td class='lessen'>10</td><td>26.9 MISC.GLUTATHIONE S TRANSFERASES</td><td><a href='26.9 MISC.GLUTATHIONE S TRANSFERASES.html'>Details ...</a></td><td>90</td><td>0.56</td><td>1.64</td><td>0.000</td><td>0.078</td><td>0.501</td><td>6404</td><td>tags=42%, list=13%, signal=48%</td></tr><tr><td class='lessen'>11</td><td>30.2.24 SIGNALLING.RECEPTOR KINASES.S-LOCUS GLYCOPROTEIN LIKE</td><td><a href='30.2.24 SIGNALLING.RECEPTOR KINASES.S-LOCUS GLYCOPROTEIN LIKE.html'>Details ...</a></td><td>87</td><td>0.57</td><td>1.64</td><td>0.003</td><td>0.071</td><td>0.501</td><td>10442</td><td>tags=47%, list=21%, signal=60%</td></tr><tr><td class='lessen'>12</td><td>27.3.23 RNA.REGULATION OF TRANSCRIPTION.HEAT-SHOCK TRANSCRIPTION FACTOR FAMILY (HSF)</td><td><a href='27.3.23 RNA.REGULATION OF TRANSCRIPTION.HEAT-SHOCK TRANSCRIPTION FACTOR FAMILY (HSF).html'>Details ...</a></td><td>33</td><td>0.68</td><td>1.63</td><td>0.005</td><td>0.069</td><td>0.519</td><td>9979</td><td>tags=58%, list=20%, signal=72%</td></tr><tr><td class='lessen'>13</td><td>29.5.11.4.5.2 PROTEIN.DEGRADATION.UBIQUITIN.E3.BTB/POZ CULLIN3.BTB/POZ</td><td><a href='29.5.11.4.5.2 PROTEIN.DEGRADATION.UBIQUITIN.E3.BTB/POZ CULLIN3.BTB/POZ.html'>Details ...</a></td><td>17</td><td>0.77</td><td>1.62</td><td>0.014</td><td>0.074</td><td>0.579</td><td>5822</td><td>tags=65%, list=12%, signal=73%</td></tr><tr><td class='lessen'>14</td><td>33.30.1 DEVELOPMENT.MULTITARGET.TARGET OF RAPAMYCIN</td><td><a href='33.30.1 DEVELOPMENT.MULTITARGET.TARGET OF RAPAMYCIN.html'>Details ...</a></td><td>20</td><td>0.75</td><td>1.62</td><td>0.010</td><td>0.070</td><td>0.587</td><td>11938</td><td>tags=35%, list=24%, signal=46%</td></tr><tr><td class='lessen'>15</td><td>29.5.2 PROTEIN.DEGRADATION.AUTOPHAGY</td><td><a href='29.5.2 PROTEIN.DEGRADATION.AUTOPHAGY.html'>Details ...</a></td><td>22</td><td>0.73</td><td>1.62</td><td>0.012</td><td>0.066</td><td>0.587</td><td>8352</td><td>tags=77%, list=17%, signal=93%</td></tr><tr><td class='lessen'>16</td><td>29.2.1.99.99 PROTEIN.SYNTHESIS.RIBOSOMAL PROTEIN.UNKNOWN.UNKNOWN</td><td><a href='29.2.1.99.99 PROTEIN.SYNTHESIS.RIBOSOMAL PROTEIN.UNKNOWN.UNKNOWN.html'>Details ...</a></td><td>68</td><td>0.58</td><td>1.61</td><td>0.005</td><td>0.063</td><td>0.595</td><td>4361</td><td>tags=9%, list=9%, signal=10%</td></tr><tr><td class='lessen'>17</td><td>30.4 SIGNALLING.PHOSPHINOSITIDES</td><td><a href='30.4 SIGNALLING.PHOSPHINOSITIDES.html'>Details ...</a></td><td>78</td><td>0.56</td><td>1.60</td><td>0.000</td><td>0.065</td><td>0.630</td><td>8660</td><td>tags=47%, list=18%, signal=57%</td></tr><tr><td class='lessen'>18</td><td>20.1.7.6 STRESS.BIOTIC.PR-PROTEINS.PR6 (PROTEINASE INHIBITORS)</td><td><a href='20.1.7.6 STRESS.BIOTIC.PR-PROTEINS.PR6 (PROTEINASE INHIBITORS).html'>Details ...</a></td><td>55</td><td>0.59</td><td>1.59</td><td>0.003</td><td>0.075</td><td>0.705</td><td>5436</td><td>tags=33%, list=11%, signal=37%</td></tr><tr><td class='lessen'>19</td><td>35.1.40 NOT ASSIGNED.NO ONTOLOGY.GLYCINE RICH PROTEINS</td><td><a href='35.1.40 NOT ASSIGNED.NO ONTOLOGY.GLYCINE RICH PROTEINS.html'>Details ...</a></td><td>46</td><td>0.61</td><td>1.58</td><td>0.005</td><td>0.076</td><td>0.729</td><td>5452</td><td>tags=35%, list=11%, signal=39%</td></tr><tr><td class='lessen'>20</td><td>13.2 AMINO ACID METABOLISM.DEGRADATION</td><td><a href='13.2 AMINO ACID METABOLISM.DEGRADATION.html'>Details ...</a></td><td>124</td><td>0.53</td><td>1.58</td><td>0.000</td><td>0.075</td><td>0.744</td><td>5498</td><td>tags=29%, list=11%, signal=33%</td></tr><tr><td class='lessen'>21</td><td>27.3.7 RNA.REGULATION OF TRANSCRIPTION.C2C2(ZN) CONSTANS-LIKE ZINC FINGER FAMILY (CO-LIKE)</td><td><a href='27.3.7 RNA.REGULATION OF TRANSCRIPTION.C2C2(ZN) CONSTANS-LIKE ZINC FINGER FAMILY (CO-LIKE).html'>Details ...</a></td><td>45</td><td>0.61</td><td>1.56</td><td>0.013</td><td>0.089</td><td>0.821</td><td>8320</td><td>tags=44%, list=17%, signal=53%</td></tr><tr><td class='lessen'>22</td><td>3.2.3 MINOR CHO METABOLISM.TREHALOSE.POTENTIAL TPS/TPP</td><td><a href='3.2.3 MINOR CHO METABOLISM.TREHALOSE.POTENTIAL TPS/TPP.html'>Details ...</a></td><td>15</td><td>0.74</td><td>1.52</td><td>0.039</td><td>0.117</td><td>0.905</td><td>2346</td><td>tags=53%, list=5%, signal=56%</td></tr><tr><td class='lessen'>23</td><td>13.2.4 AMINO ACID METABOLISM.DEGRADATION.BRANCHED CHAIN GROUP</td><td><a href='13.2.4 AMINO ACID METABOLISM.DEGRADATION.BRANCHED CHAIN GROUP.html'>Details ...</a></td><td>16</td><td>0.74</td><td>1.50</td><td>0.038</td><td>0.140</td><td>0.953</td><td>6681</td><td>tags=69%, list=14%, signal=79%</td></tr><tr><td class='lessen'>24</td><td>30.2.8.2 SIGNALLING.RECEPTOR KINASES.LEUCINE RICH REPEAT VIII.TYPE 2</td><td><a href='30.2.8.2 SIGNALLING.RECEPTOR KINASES.LEUCINE RICH REPEAT VIII.TYPE 2.html'>Details ...</a></td><td>41</td><td>0.59</td><td>1.49</td><td>0.018</td><td>0.142</td><td>0.960</td><td>11369</td><td>tags=59%, list=23%, signal=76%</td></tr><tr><td class='lessen'>25</td><td>35.1.19 NOT ASSIGNED.NO ONTOLOGY.C2 DOMAIN-CONTAINING PROTEIN</td><td><a href='35.1.19 NOT ASSIGNED.NO ONTOLOGY.C2 DOMAIN-CONTAINING PROTEIN.html'>Details ...</a></td><td>19</td><td>0.69</td><td>1.48</td><td>0.035</td><td>0.145</td><td>0.972</td><td>9653</td><td>tags=58%, list=20%, signal=72%</td></tr><tr><td class='lessen'>26</td><td>26.16 MISC.MYROSINASES-LECTIN-JACALIN</td><td><a href='26.16 MISC.MYROSINASES-LECTIN-JACALIN.html'>Details ...</a></td><td>33</td><td>0.60</td><td>1.48</td><td>0.025</td><td>0.140</td><td>0.972</td><td>3979</td><td>tags=21%, list=8%, signal=23%</td></tr><tr><td class='lessen'>27</td><td>27.3.27 RNA.REGULATION OF TRANSCRIPTION.NAC DOMAIN TRANSCRIPTION FACTOR FAMILY</td><td><a href='27.3.27 RNA.REGULATION OF TRANSCRIPTION.NAC DOMAIN TRANSCRIPTION FACTOR FAMILY.html'>Details ...</a></td><td>86</td><td>0.51</td><td>1.47</td><td>0.003</td><td>0.147</td><td>0.979</td><td>6151</td><td>tags=42%, list=12%, signal=48%</td></tr><tr><td class='lessen'>28</td><td>30.2.8 SIGNALLING.RECEPTOR KINASES.LEUCINE RICH REPEAT VIII</td><td><a href='30.2.8 SIGNALLING.RECEPTOR KINASES.LEUCINE RICH REPEAT VIII.html'>Details ...</a></td><td>62</td><td>0.55</td><td>1.47</td><td>0.016</td><td>0.147</td><td>0.979</td><td>4627</td><td>tags=29%, list=9%, signal=32%</td></tr><tr><td class='lessen'>29</td><td>35.1.9 NOT ASSIGNED.NO ONTOLOGY.BTB/POZ DOMAIN-CONTAINING PROTEIN</td><td><a href='35.1.9 NOT ASSIGNED.NO ONTOLOGY.BTB/POZ DOMAIN-CONTAINING PROTEIN.html'>Details ...</a></td><td>15</td><td>0.71</td><td>1.46</td><td>0.029</td><td>0.159</td><td>0.983</td><td>6893</td><td>tags=40%, list=14%, signal=46%</td></tr><tr><td class='lessen'>30</td><td>17.5.1.1 HORMONE METABOLISM.ETHYLENE.SYNTHESIS-DEGRADATION.1-AMINOCYCLOPROPANE-1-CARBOXYLATE SYNTHASE</td><td><a href='17.5.1.1 HORMONE METABOLISM.ETHYLENE.SYNTHESIS-DEGRADATION.1-AMINOCYCLOPROPANE-1-CARBOXYLATE SYNTHASE.html'>Details ...</a></td><td>20</td><td>0.68</td><td>1.45</td><td>0.054</td><td>0.158</td><td>0.986</td><td>9667</td><td>tags=55%, list=20%, signal=68%</td></tr><tr><td class='lessen'>31</td><td>20.2.1 STRESS.ABIOTIC.HEAT</td><td><a href='20.2.1 STRESS.ABIOTIC.HEAT.html'>Details ...</a></td><td>372</td><td>0.42</td><td>1.45</td><td>0.000</td><td>0.153</td><td>0.986</td><td>10310</td><td>tags=34%, list=21%, signal=42%</td></tr><tr><td class='lessen'>32</td><td>27.3.24 RNA.REGULATION OF TRANSCRIPTION.MADS BOX TRANSCRIPTION FACTOR FAMILY</td><td><a href='27.3.24 RNA.REGULATION OF TRANSCRIPTION.MADS BOX TRANSCRIPTION FACTOR FAMILY.html'>Details ...</a></td><td>177</td><td>0.46</td><td>1.44</td><td>0.000</td><td>0.157</td><td>0.988</td><td>9947</td><td>tags=19%, list=20%, signal=23%</td></tr><tr><td class='lessen'>33</td><td>27.3.64 RNA.REGULATION OF TRANSCRIPTION.PHOR1</td><td><a href='27.3.64 RNA.REGULATION OF TRANSCRIPTION.PHOR1.html'>Details ...</a></td><td>21</td><td>0.65</td><td>1.44</td><td>0.048</td><td>0.162</td><td>0.995</td><td>8265</td><td>tags=48%, list=17%, signal=57%</td></tr><tr><td class='lessen'>34</td><td>11.9.3.1 LIPID METABOLISM.LIPID DEGRADATION.LYSOPHOSPHOLIPASES.PHOSPHOLIPASE D</td><td><a href='11.9.3.1 LIPID METABOLISM.LIPID DEGRADATION.LYSOPHOSPHOLIPASES.PHOSPHOLIPASE D.html'>Details ...</a></td><td>25</td><td>0.62</td><td>1.44</td><td>0.044</td><td>0.158</td><td>0.995</td><td>12138</td><td>tags=36%, list=25%, signal=48%</td></tr><tr><td class='lessen'>35</td><td>34.14 TRANSPORT.UNSPECIFIED CATIONS</td><td><a href='34.14 TRANSPORT.UNSPECIFIED CATIONS.html'>Details ...</a></td><td>68</td><td>0.51</td><td>1.43</td><td>0.008</td><td>0.167</td><td>0.996</td><td>6163</td><td>tags=26%, list=12%, signal=30%</td></tr><tr><td class='lessen'>36</td><td>29.5.11.4.3.3 PROTEIN.DEGRADATION.UBIQUITIN.E3.SCF.CULLIN</td><td><a href='29.5.11.4.3.3 PROTEIN.DEGRADATION.UBIQUITIN.E3.SCF.CULLIN.html'>Details ...</a></td><td>36</td><td>0.58</td><td>1.41</td><td>0.039</td><td>0.184</td><td>0.997</td><td>8783</td><td>tags=31%, list=18%, signal=37%</td></tr><tr><td class='lessen'>37</td><td>26.24 MISC.GCN5-RELATED N-ACETYLTRANSFERASE</td><td><a href='26.24 MISC.GCN5-RELATED N-ACETYLTRANSFERASE.html'>Details ...</a></td><td>58</td><td>0.52</td><td>1.41</td><td>0.035</td><td>0.183</td><td>0.997</td><td>4525</td><td>tags=41%, list=9%, signal=46%</td></tr><tr><td class='lessen'>38</td><td>17.1.3 HORMONE METABOLISM.ABSCISIC ACID.INDUCED-REGULATED-RESPONSIVE-ACTIVATED</td><td><a href='17.1.3 HORMONE METABOLISM.ABSCISIC ACID.INDUCED-REGULATED-RESPONSIVE-ACTIVATED.html'>Details ...</a></td><td>32</td><td>0.59</td><td>1.39</td><td>0.075</td><td>0.195</td><td>0.997</td><td>7245</td><td>tags=59%, list=15%, signal=70%</td></tr><tr><td class='lessen'>39</td><td>29.4.1.57 PROTEIN.POSTRANSLATIONAL MODIFICATION.KINASE.RECEPTOR LIKE CYTOPLASMATIC KINASE VII</td><td><a href='29.4.1.57 PROTEIN.POSTRANSLATIONAL MODIFICATION.KINASE.RECEPTOR LIKE CYTOPLASMATIC KINASE VII.html'>Details ...</a></td><td>59</td><td>0.51</td><td>1.39</td><td>0.020</td><td>0.192</td><td>0.997</td><td>6232</td><td>tags=37%, list=13%, signal=43%</td></tr><tr><td class='lessen'>40</td><td>11.9.4 LIPID METABOLISM.LIPID DEGRADATION.BETA-OXIDATION</td><td><a href='11.9.4 LIPID METABOLISM.LIPID DEGRADATION.BETA-OXIDATION.html'>Details ...</a></td><td>70</td><td>0.50</td><td>1.39</td><td>0.027</td><td>0.193</td><td>0.997</td><td>5164</td><td>tags=23%, list=10%, signal=25%</td></tr><tr><td class='lessen'>41</td><td>17.5 HORMONE METABOLISM.ETHYLENE</td><td><a href='17.5 HORMONE METABOLISM.ETHYLENE.html'>Details ...</a></td><td>238</td><td>0.42</td><td>1.38</td><td>0.004</td><td>0.200</td><td>0.997</td><td>5429</td><td>tags=21%, list=11%, signal=23%</td></tr><tr><td class='lessen'>42</td><td>5 FERMENTATION</td><td><a href='5 FERMENTATION.html'>Details ...</a></td><td>44</td><td>0.53</td><td>1.36</td><td>0.061</td><td>0.217</td><td>0.999</td><td>8893</td><td>tags=36%, list=18%, signal=44%</td></tr><tr><td class='lessen'>43</td><td>34.4 TRANSPORT.NITRATE</td><td><a href='34.4 TRANSPORT.NITRATE.html'>Details ...</a></td><td>29</td><td>0.58</td><td>1.36</td><td>0.070</td><td>0.213</td><td>0.999</td><td>4313</td><td>tags=34%, list=9%, signal=38%</td></tr><tr><td class='lessen'>44</td><td>16.2.1 SECONDARY METABOLISM.PHENYLPROPANOIDS.LIGNIN BIOSYNTHESIS</td><td><a href='16.2.1 SECONDARY METABOLISM.PHENYLPROPANOIDS.LIGNIN BIOSYNTHESIS.html'>Details ...</a></td><td>66</td><td>0.50</td><td>1.36</td><td>0.031</td><td>0.208</td><td>0.999</td><td>6629</td><td>tags=27%, list=13%, signal=31%</td></tr><tr><td class='lessen'>45</td><td>20.2 STRESS.ABIOTIC</td><td><a href='20.2 STRESS.ABIOTIC.html'>Details ...</a></td><td>496</td><td>0.39</td><td>1.34</td><td>0.000</td><td>0.235</td><td>1.000</td><td>10181</td><td>tags=34%, list=21%, signal=42%</td></tr><tr><td class='lessen'>46</td><td>30.6 SIGNALLING.MAP KINASES</td><td><a href='30.6 SIGNALLING.MAP KINASES.html'>Details ...</a></td><td>112</td><td>0.44</td><td>1.34</td><td>0.028</td><td>0.236</td><td>1.000</td><td>10638</td><td>tags=35%, list=22%, signal=44%</td></tr><tr><td class='lessen'>47</td><td>6 GLUCONEOGENESIS / GLYOXYLATE CYCLE</td><td><a href='6 GLUCONEOGENESIS / GLYOXYLATE CYCLE.html'>Details ...</a></td><td>17</td><td>0.64</td><td>1.33</td><td>0.133</td><td>0.241</td><td>1.000</td><td>5048</td><td>tags=29%, list=10%, signal=33%</td></tr><tr><td class='lessen'>48</td><td>35.1.21 NOT ASSIGNED.NO ONTOLOGY.EPSIN N-TERMINAL HOMOLOGY (ENTH) DOMAIN-CONTAINING PROTEIN</td><td><a href='35.1.21 NOT ASSIGNED.NO ONTOLOGY.EPSIN N-TERMINAL HOMOLOGY (ENTH) DOMAIN-CONTAINING PROTEIN.html'>Details ...</a></td><td>26</td><td>0.57</td><td>1.33</td><td>0.098</td><td>0.247</td><td>1.000</td><td>7256</td><td>tags=35%, list=15%, signal=41%</td></tr><tr><td class='lessen'>49</td><td>16.2.1.10 SECONDARY METABOLISM.PHENYLPROPANOIDS.LIGNIN BIOSYNTHESIS.CAD</td><td><a href='16.2.1.10 SECONDARY METABOLISM.PHENYLPROPANOIDS.LIGNIN BIOSYNTHESIS.CAD.html'>Details ...</a></td><td>33</td><td>0.54</td><td>1.32</td><td>0.085</td><td>0.246</td><td>1.000</td><td>3943</td><td>tags=21%, list=8%, signal=23%</td></tr><tr><td class='lessen'>50</td><td>28.1.1 DNA.SYNTHESIS/CHROMATIN STRUCTURE.RETROTRANSPOSON/TRANSPOSASE</td><td><a href='28.1.1 DNA.SYNTHESIS/CHROMATIN STRUCTURE.RETROTRANSPOSON/TRANSPOSASE.html'>Details ...</a></td><td>236</td><td>0.40</td><td>1.32</td><td>0.000</td><td>0.251</td><td>1.000</td><td>9916</td><td>tags=12%, list=20%, signal=15%</td></tr><tr><td class='lessen'>51</td><td>17.5.2 HORMONE METABOLISM.ETHYLENE.SIGNAL TRANSDUCTION</td><td></td><td>100</td><td>0.44</td><td>1.31</td><td>0.047</td><td>0.264</td><td>1.000</td><td>4613</td><td>tags=20%, list=9%, signal=22%</td></tr><tr><td class='lessen'>52</td><td>27.3.57 RNA.REGULATION OF TRANSCRIPTION.JUMONJI FAMILY</td><td></td><td>32</td><td>0.54</td><td>1.31</td><td>0.082</td><td>0.261</td><td>1.000</td><td>10153</td><td>tags=44%, list=21%, signal=55%</td></tr><tr><td class='lessen'>53</td><td>17.5.1 HORMONE METABOLISM.ETHYLENE.SYNTHESIS-DEGRADATION</td><td></td><td>126</td><td>0.42</td><td>1.28</td><td>0.033</td><td>0.305</td><td>1.000</td><td>5323</td><td>tags=20%, list=11%, signal=22%</td></tr><tr><td class='lessen'>54</td><td>11.3 LIPID METABOLISM.PHOSPHOLIPID SYNTHESIS</td><td></td><td>65</td><td>0.45</td><td>1.26</td><td>0.083</td><td>0.331</td><td>1.000</td><td>7328</td><td>tags=26%, list=15%, signal=31%</td></tr><tr><td class='lessen'>55</td><td>27.3.30 RNA.REGULATION OF TRANSCRIPTION.TRIPLE-HELIX TRANSCRIPTION FACTOR FAMILY (TRIHELIX)</td><td></td><td>32</td><td>0.51</td><td>1.26</td><td>0.091</td><td>0.327</td><td>1.000</td><td>7025</td><td>tags=47%, list=14%, signal=55%</td></tr><tr><td class='lessen'>56</td><td>29.2.1.99 PROTEIN.SYNTHESIS.RIBOSOMAL PROTEIN.UNKNOWN</td><td></td><td>74</td><td>0.45</td><td>1.26</td><td>0.056</td><td>0.326</td><td>1.000</td><td>4361</td><td>tags=8%, list=9%, signal=9%</td></tr><tr><td class='lessen'>57</td><td>27.3.44 RNA.REGULATION OF TRANSCRIPTION.CHROMATIN REMODELING FACTORS</td><td></td><td>63</td><td>0.46</td><td>1.25</td><td>0.086</td><td>0.334</td><td>1.000</td><td>8870</td><td>tags=41%, list=18%, signal=50%</td></tr><tr><td class='lessen'>58</td><td>20.1.2.2 STRESS.BIOTIC.RECEPTORS.TIR-NBS-LRR</td><td></td><td>140</td><td>0.40</td><td>1.25</td><td>0.040</td><td>0.335</td><td>1.000</td><td>10300</td><td>tags=37%, list=21%, signal=47%</td></tr><tr><td class='lessen'>59</td><td>30.4.1 SIGNALLING.PHOSPHINOSITIDES.PHOSPHATIDYLINOSITOL-4-PHOSPHATE 5-KINASE</td><td></td><td>24</td><td>0.55</td><td>1.25</td><td>0.137</td><td>0.335</td><td>1.000</td><td>8660</td><td>tags=54%, list=18%, signal=66%</td></tr><tr><td class='lessen'>60</td><td>33.1 DEVELOPMENT.STORAGE PROTEINS</td><td></td><td>66</td><td>0.45</td><td>1.24</td><td>0.090</td><td>0.348</td><td>1.000</td><td>3892</td><td>tags=21%, list=8%, signal=23%</td></tr><tr><td class='lessen'>61</td><td>17.1 HORMONE METABOLISM.ABSCISIC ACID</td><td></td><td>111</td><td>0.41</td><td>1.24</td><td>0.070</td><td>0.344</td><td>1.000</td><td>6153</td><td>tags=30%, list=12%, signal=34%</td></tr><tr><td class='lessen'>62</td><td>13.2.3 AMINO ACID METABOLISM.DEGRADATION.ASPARTATE FAMILY</td><td></td><td>24</td><td>0.54</td><td>1.24</td><td>0.172</td><td>0.339</td><td>1.000</td><td>4051</td><td>tags=29%, list=8%, signal=32%</td></tr><tr><td class='lessen'>63</td><td>16.1.4 SECONDARY METABOLISM.ISOPRENOIDS.CAROTENOIDS</td><td></td><td>33</td><td>0.51</td><td>1.23</td><td>0.141</td><td>0.351</td><td>1.000</td><td>11036</td><td>tags=48%, list=22%, signal=62%</td></tr><tr><td class='lessen'>64</td><td>31.5.1 CELL.CELL DEATH.PLANTS</td><td></td><td>33</td><td>0.50</td><td>1.21</td><td>0.170</td><td>0.388</td><td>1.000</td><td>5428</td><td>tags=15%, list=11%, signal=17%</td></tr><tr><td class='lessen'>65</td><td>16.1.5 SECONDARY METABOLISM.ISOPRENOIDS.TERPENOIDS</td><td></td><td>155</td><td>0.38</td><td>1.20</td><td>0.071</td><td>0.406</td><td>1.000</td><td>8008</td><td>tags=13%, list=16%, signal=15%</td></tr><tr><td class='lessen'>66</td><td>29.5.11.4.3.1 PROTEIN.DEGRADATION.UBIQUITIN.E3.SCF.SKP</td><td></td><td>56</td><td>0.44</td><td>1.20</td><td>0.144</td><td>0.406</td><td>1.000</td><td>12195</td><td>tags=13%, list=25%, signal=17%</td></tr><tr><td class='lessen'>67</td><td>34.22 TRANSPORT.CYCLIC NUCLEOTIDE OR CALCIUM REGULATED CHANNELS</td><td></td><td>26</td><td>0.51</td><td>1.19</td><td>0.204</td><td>0.427</td><td>1.000</td><td>4891</td><td>tags=31%, list=10%, signal=34%</td></tr><tr><td class='lessen'>68</td><td>27.3.63 RNA.REGULATION OF TRANSCRIPTION.PHD FINGER TRANSCRIPTION FACTOR</td><td></td><td>49</td><td>0.44</td><td>1.17</td><td>0.172</td><td>0.451</td><td>1.000</td><td>11614</td><td>tags=45%, list=24%, signal=59%</td></tr><tr><td class='lessen'>69</td><td>23.3.3 NUCLEOTIDE METABOLISM.SALVAGE.NUDIX HYDROLASES</td><td></td><td>36</td><td>0.48</td><td>1.17</td><td>0.196</td><td>0.449</td><td>1.000</td><td>8427</td><td>tags=31%, list=17%, signal=37%</td></tr><tr><td class='lessen'>70</td><td>18.4 CO-FACTOR AND VITAMINE METABOLISM.PANTOTHENATE</td><td></td><td>16</td><td>0.57</td><td>1.17</td><td>0.255</td><td>0.450</td><td>1.000</td><td>10761</td><td>tags=69%, list=22%, signal=88%</td></tr><tr><td class='lessen'>71</td><td>29.5.3 PROTEIN.DEGRADATION.CYSTEINE PROTEASE</td><td></td><td>190</td><td>0.36</td><td>1.17</td><td>0.085</td><td>0.445</td><td>1.000</td><td>10293</td><td>tags=21%, list=21%, signal=27%</td></tr><tr><td class='lessen'>72</td><td>27.3.11 RNA.REGULATION OF TRANSCRIPTION.C2H2 ZINC FINGER FAMILY</td><td></td><td>226</td><td>0.36</td><td>1.16</td><td>0.063</td><td>0.448</td><td>1.000</td><td>10446</td><td>tags=42%, list=21%, signal=53%</td></tr><tr><td class='lessen'>73</td><td>16.1 SECONDARY METABOLISM.ISOPRENOIDS</td><td></td><td>268</td><td>0.35</td><td>1.15</td><td>0.071</td><td>0.471</td><td>1.000</td><td>8008</td><td>tags=16%, list=16%, signal=19%</td></tr><tr><td class='lessen'>74</td><td>2.2.2.1 MAJOR CHO METABOLISM.DEGRADATION.STARCH.STARCH CLEAVAGE</td><td></td><td>18</td><td>0.55</td><td>1.15</td><td>0.274</td><td>0.467</td><td>1.000</td><td>5899</td><td>tags=33%, list=12%, signal=38%</td></tr><tr><td class='lessen'>75</td><td>30.2.17 SIGNALLING.RECEPTOR KINASES.DUF 26</td><td></td><td>94</td><td>0.40</td><td>1.15</td><td>0.150</td><td>0.466</td><td>1.000</td><td>8471</td><td>tags=33%, list=17%, signal=40%</td></tr><tr><td class='lessen'>76</td><td>27.3.60 RNA.REGULATION OF TRANSCRIPTION.NIN-LIKE BZIP-RELATED FAMILY</td><td></td><td>29</td><td>0.49</td><td>1.15</td><td>0.217</td><td>0.466</td><td>1.000</td><td>9583</td><td>tags=38%, list=19%, signal=47%</td></tr><tr><td class='lessen'>77</td><td>13.1.1 AMINO ACID METABOLISM.SYNTHESIS.CENTRAL AMINO ACID METABOLISM</td><td></td><td>24</td><td>0.51</td><td>1.15</td><td>0.224</td><td>0.466</td><td>1.000</td><td>2397</td><td>tags=29%, list=5%, signal=31%</td></tr><tr><td class='lessen'>78</td><td>3.2 MINOR CHO METABOLISM.TREHALOSE</td><td></td><td>27</td><td>0.50</td><td>1.14</td><td>0.242</td><td>0.462</td><td>1.000</td><td>3764</td><td>tags=37%, list=8%, signal=40%</td></tr><tr><td class='lessen'>79</td><td>11.8.1 LIPID METABOLISM.EXOTICS (STEROIDS, SQUALENE ETC).SPHINGOLIPIDS</td><td></td><td>43</td><td>0.44</td><td>1.14</td><td>0.206</td><td>0.478</td><td>1.000</td><td>7637</td><td>tags=37%, list=15%, signal=44%</td></tr><tr><td class='lessen'>80</td><td>34.15 TRANSPORT.POTASSIUM</td><td></td><td>72</td><td>0.41</td><td>1.13</td><td>0.211</td><td>0.479</td><td>1.000</td><td>5664</td><td>tags=21%, list=11%, signal=24%</td></tr><tr><td class='lessen'>81</td><td>29.5.7 PROTEIN.DEGRADATION.METALLOPROTEASE</td><td></td><td>52</td><td>0.43</td><td>1.13</td><td>0.249</td><td>0.476</td><td>1.000</td><td>10830</td><td>tags=37%, list=22%, signal=47%</td></tr><tr><td class='lessen'>82</td><td>16.8 SECONDARY METABOLISM.FLAVONOIDS</td><td></td><td>238</td><td>0.34</td><td>1.13</td><td>0.109</td><td>0.477</td><td>1.000</td><td>4261</td><td>tags=15%, list=9%, signal=16%</td></tr><tr><td class='lessen'>83</td><td>11.9 LIPID METABOLISM.LIPID DEGRADATION</td><td></td><td>220</td><td>0.34</td><td>1.12</td><td>0.125</td><td>0.493</td><td>1.000</td><td>8269</td><td>tags=29%, list=17%, signal=34%</td></tr><tr><td class='lessen'>84</td><td>30.3 SIGNALLING.CALCIUM</td><td></td><td>395</td><td>0.33</td><td>1.12</td><td>0.073</td><td>0.504</td><td>1.000</td><td>7254</td><td>tags=25%, list=15%, signal=29%</td></tr><tr><td class='lessen'>85</td><td>1.1.6 PS.LIGHTREACTION.NADH DH</td><td></td><td>42</td><td>0.43</td><td>1.11</td><td>0.251</td><td>0.512</td><td>1.000</td><td>8897</td><td>tags=21%, list=18%, signal=26%</td></tr><tr><td class='lessen'>86</td><td>29.3.4.3 PROTEIN.TARGETING.SECRETORY PATHWAY.VACUOLE</td><td></td><td>44</td><td>0.42</td><td>1.11</td><td>0.267</td><td>0.511</td><td>1.000</td><td>12736</td><td>tags=41%, list=26%, signal=55%</td></tr><tr><td class='lessen'>87</td><td>10.8.2 CELL WALL.PECTINESTERASES.ACETYL ESTERASE</td><td></td><td>21</td><td>0.50</td><td>1.10</td><td>0.301</td><td>0.515</td><td>1.000</td><td>4615</td><td>tags=24%, list=9%, signal=26%</td></tr><tr><td class='lessen'>88</td><td>20.1.3.1 STRESS.BIOTIC.SIGNALLING.MLO-LIKE</td><td></td><td>21</td><td>0.49</td><td>1.10</td><td>0.305</td><td>0.512</td><td>1.000</td><td>4323</td><td>tags=24%, list=9%, signal=26%</td></tr><tr><td class='lessen'>89</td><td>27.3.35 RNA.REGULATION OF TRANSCRIPTION.BZIP TRANSCRIPTION FACTOR FAMILY</td><td></td><td>142</td><td>0.36</td><td>1.10</td><td>0.201</td><td>0.516</td><td>1.000</td><td>9756</td><td>tags=36%, list=20%, signal=45%</td></tr><tr><td class='lessen'>90</td><td>11.1.8 LIPID METABOLISM.FA SYNTHESIS AND FA ELONGATION.ACYL COA LIGASE</td><td></td><td>34</td><td>0.45</td><td>1.10</td><td>0.307</td><td>0.519</td><td>1.000</td><td>6331</td><td>tags=29%, list=13%, signal=34%</td></tr><tr><td class='lessen'>91</td><td>34.16 TRANSPORT.ABC TRANSPORTERS AND MULTIDRUG RESISTANCE SYSTEMS</td><td></td><td>219</td><td>0.34</td><td>1.09</td><td>0.185</td><td>0.519</td><td>1.000</td><td>7617</td><td>tags=27%, list=15%, signal=32%</td></tr><tr><td class='lessen'>92</td><td>13.1.5.3 AMINO ACID METABOLISM.SYNTHESIS.SERINE-GLYCINE-CYSTEINE GROUP.CYSTEINE</td><td></td><td>32</td><td>0.45</td><td>1.09</td><td>0.270</td><td>0.518</td><td>1.000</td><td>9836</td><td>tags=34%, list=20%, signal=43%</td></tr><tr><td class='lessen'>93</td><td>17.1.2 HORMONE METABOLISM.ABSCISIC ACID.SIGNAL TRANSDUCTION</td><td></td><td>33</td><td>0.44</td><td>1.08</td><td>0.316</td><td>0.538</td><td>1.000</td><td>5492</td><td>tags=21%, list=11%, signal=24%</td></tr><tr><td class='lessen'>94</td><td>27.3.25 RNA.REGULATION OF TRANSCRIPTION.MYB DOMAIN TRANSCRIPTION FACTOR FAMILY</td><td></td><td>235</td><td>0.33</td><td>1.08</td><td>0.197</td><td>0.538</td><td>1.000</td><td>6562</td><td>tags=25%, list=13%, signal=29%</td></tr><tr><td class='lessen'>95</td><td>34.2 TRANSPORT.SUGARS</td><td></td><td>116</td><td>0.36</td><td>1.08</td><td>0.245</td><td>0.535</td><td>1.000</td><td>6468</td><td>tags=24%, list=13%, signal=28%</td></tr><tr><td class='lessen'>96</td><td>29.2.5 PROTEIN.SYNTHESIS.RELEASE</td><td></td><td>17</td><td>0.50</td><td>1.05</td><td>0.393</td><td>0.604</td><td>1.000</td><td>9390</td><td>tags=47%, list=19%, signal=58%</td></tr><tr><td class='lessen'>97</td><td>27.3.67 RNA.REGULATION OF TRANSCRIPTION.PUTATIVE TRANSCRIPTION REGULATOR</td><td></td><td>217</td><td>0.32</td><td>1.05</td><td>0.256</td><td>0.598</td><td>1.000</td><td>7708</td><td>tags=33%, list=16%, signal=39%</td></tr><tr><td class='lessen'>98</td><td>17.8 HORMONE METABOLISM.SALICYLIC ACID</td><td></td><td>22</td><td>0.47</td><td>1.05</td><td>0.357</td><td>0.596</td><td>1.000</td><td>8687</td><td>tags=23%, list=18%, signal=28%</td></tr><tr><td class='lessen'>99</td><td>23.3 NUCLEOTIDE METABOLISM.SALVAGE</td><td></td><td>60</td><td>0.38</td><td>1.05</td><td>0.339</td><td>0.599</td><td>1.000</td><td>7291</td><td>tags=28%, list=15%, signal=33%</td></tr><tr><td class='lessen'>100</td><td>29.4.1.53 PROTEIN.POSTRANSLATIONAL MODIFICATION.KINASE.RECEPTOR LIKE CYTOPLASMATIC KINASE III</td><td></td><td>19</td><td>0.50</td><td>1.05</td><td>0.372</td><td>0.595</td><td>1.000</td><td>4208</td><td>tags=16%, list=9%, signal=17%</td></tr><tr><td class='lessen'>101</td><td>1.1.40 PS.LIGHTREACTION.CYCLIC ELECTRON FLOW-CHLORORESPIRATION</td><td></td><td>28</td><td>0.45</td><td>1.05</td><td>0.365</td><td>0.592</td><td>1.000</td><td>5696</td><td>tags=18%, list=12%, signal=20%</td></tr><tr><td class='lessen'>102</td><td>20.1.7 STRESS.BIOTIC.PR-PROTEINS</td><td></td><td>208</td><td>0.33</td><td>1.05</td><td>0.291</td><td>0.587</td><td>1.000</td><td>6510</td><td>tags=26%, list=13%, signal=30%</td></tr><tr><td class='lessen'>103</td><td>16.1.1 SECONDARY METABOLISM.ISOPRENOIDS.NON-MEVALONATE PATHWAY</td><td></td><td>23</td><td>0.47</td><td>1.05</td><td>0.366</td><td>0.583</td><td>1.000</td><td>1464</td><td>tags=17%, list=3%, signal=18%</td></tr><tr><td class='lessen'>104</td><td>27.3.69 RNA.REGULATION OF TRANSCRIPTION.SET-DOMAIN TRANSCRIPTIONAL REGULATOR FAMILY</td><td></td><td>73</td><td>0.37</td><td>1.04</td><td>0.353</td><td>0.601</td><td>1.000</td><td>6399</td><td>tags=21%, list=13%, signal=24%</td></tr><tr><td class='lessen'>105</td><td>27.3.41 RNA.REGULATION OF TRANSCRIPTION.B3 TRANSCRIPTION FACTOR FAMILY</td><td></td><td>40</td><td>0.41</td><td>1.04</td><td>0.364</td><td>0.602</td><td>1.000</td><td>8789</td><td>tags=25%, list=18%, signal=30%</td></tr><tr><td class='lessen'>106</td><td>28.1.1.4 DNA.SYNTHESIS/CHROMATIN STRUCTURE.RETROTRANSPOSON/TRANSPOSASE.HAT-LIKE TRANSPOSASE</td><td></td><td>151</td><td>0.33</td><td>1.02</td><td>0.358</td><td>0.642</td><td>1.000</td><td>10528</td><td>tags=11%, list=21%, signal=13%</td></tr><tr><td class='lessen'>107</td><td>20.1.3 STRESS.BIOTIC.SIGNALLING</td><td></td><td>57</td><td>0.38</td><td>1.02</td><td>0.397</td><td>0.640</td><td>1.000</td><td>8777</td><td>tags=28%, list=18%, signal=34%</td></tr><tr><td class='lessen'>108</td><td>2.2.1.3 MAJOR CHO METABOLISM.DEGRADATION.SUCROSE.INVERTASES</td><td></td><td>29</td><td>0.43</td><td>1.02</td><td>0.410</td><td>0.639</td><td>1.000</td><td>10291</td><td>tags=41%, list=21%, signal=52%</td></tr><tr><td class='lessen'>109</td><td>31.4 CELL.VESICLE TRANSPORT</td><td></td><td>203</td><td>0.32</td><td>1.02</td><td>0.393</td><td>0.638</td><td>1.000</td><td>10381</td><td>tags=44%, list=21%, signal=55%</td></tr><tr><td class='lessen'>110</td><td>30.2.20 SIGNALLING.RECEPTOR KINASES.WHEAT LRK10 LIKE</td><td></td><td>21</td><td>0.45</td><td>1.02</td><td>0.399</td><td>0.634</td><td>1.000</td><td>9799</td><td>tags=43%, list=20%, signal=53%</td></tr><tr><td class='lessen'>111</td><td>29.4.1 PROTEIN.POSTRANSLATIONAL MODIFICATION.KINASE</td><td></td><td>458</td><td>0.29</td><td>1.01</td><td>0.351</td><td>0.638</td><td>1.000</td><td>10483</td><td>tags=34%, list=21%, signal=43%</td></tr><tr><td class='lessen'>112</td><td>13 AMINO ACID METABOLISM</td><td></td><td>372</td><td>0.30</td><td>1.01</td><td>0.352</td><td>0.637</td><td>1.000</td><td>5956</td><td>tags=20%, list=12%, signal=23%</td></tr><tr><td class='lessen'>113</td><td>34.21 TRANSPORT.CALCIUM</td><td></td><td>24</td><td>0.44</td><td>1.01</td><td>0.438</td><td>0.645</td><td>1.000</td><td>1445</td><td>tags=17%, list=3%, signal=17%</td></tr><tr><td class='lessen'>114</td><td>27.3.5 RNA.REGULATION OF TRANSCRIPTION.ARR</td><td></td><td>38</td><td>0.40</td><td>1.00</td><td>0.457</td><td>0.659</td><td>1.000</td><td>6400</td><td>tags=18%, list=13%, signal=21%</td></tr><tr><td class='lessen'>115</td><td>27.3.54 RNA.REGULATION OF TRANSCRIPTION.HISTONE ACETYLTRANSFERASES</td><td></td><td>18</td><td>0.47</td><td>1.00</td><td>0.453</td><td>0.665</td><td>1.000</td><td>10024</td><td>tags=44%, list=20%, signal=56%</td></tr><tr><td class='lessen'>116</td><td>13.2.6 AMINO ACID METABOLISM.DEGRADATION.AROMATIC AA</td><td></td><td>29</td><td>0.42</td><td>1.00</td><td>0.483</td><td>0.660</td><td>1.000</td><td>3468</td><td>tags=21%, list=7%, signal=22%</td></tr><tr><td class='lessen'>117</td><td>17.7 HORMONE METABOLISM.JASMONATE</td><td></td><td>64</td><td>0.36</td><td>0.99</td><td>0.405</td><td>0.657</td><td>1.000</td><td>4815</td><td>tags=16%, list=10%, signal=17%</td></tr><tr><td class='lessen'>118</td><td>29.5.11.3 PROTEIN.DEGRADATION.UBIQUITIN.E2</td><td></td><td>58</td><td>0.36</td><td>0.99</td><td>0.460</td><td>0.653</td><td>1.000</td><td>11620</td><td>tags=45%, list=24%, signal=59%</td></tr><tr><td class='lessen'>119</td><td>29.2.2.3.5 PROTEIN.SYNTHESIS.RIBOSOME BIOGENESIS.PRE-RRNA PROCESSING AND MODIFICATIONS.DEXD-BOX HELICASES</td><td></td><td>20</td><td>0.46</td><td>0.99</td><td>0.486</td><td>0.665</td><td>1.000</td><td>703</td><td>tags=10%, list=1%, signal=10%</td></tr><tr><td class='lessen'>120</td><td>27.3.52 RNA.REGULATION OF TRANSCRIPTION.GLOBAL TRANSCRIPTION FACTOR GROUP</td><td></td><td>22</td><td>0.45</td><td>0.99</td><td>0.441</td><td>0.662</td><td>1.000</td><td>6729</td><td>tags=32%, list=14%, signal=37%</td></tr><tr><td class='lessen'>121</td><td>17.2.3 HORMONE METABOLISM.AUXIN.INDUCED-REGULATED-RESPONSIVE-ACTIVATED</td><td></td><td>287</td><td>0.30</td><td>0.99</td><td>0.473</td><td>0.662</td><td>1.000</td><td>8467</td><td>tags=22%, list=17%, signal=27%</td></tr><tr><td class='lessen'>122</td><td>34.9 TRANSPORT.METABOLITE TRANSPORTERS AT THE MITOCHONDRIAL MEMBRANE</td><td></td><td>100</td><td>0.33</td><td>0.98</td><td>0.461</td><td>0.666</td><td>1.000</td><td>6868</td><td>tags=31%, list=14%, signal=36%</td></tr><tr><td class='lessen'>123</td><td>24.2 BIODEGRADATION OF XENOBIOTICS.LACTOYLGLUTATHIONE LYASE</td><td></td><td>19</td><td>0.45</td><td>0.98</td><td>0.461</td><td>0.665</td><td>1.000</td><td>5739</td><td>tags=32%, list=12%, signal=36%</td></tr><tr><td class='lessen'>124</td><td>26.8 MISC.NITRILASES, NITRILE LYASES, BERBERINE BRIDGE ENZYMES, RETICULINE OXIDASES, TROPONINE REDUCTASES</td><td></td><td>136</td><td>0.31</td><td>0.97</td><td>0.506</td><td>0.693</td><td>1.000</td><td>3502</td><td>tags=14%, list=7%, signal=15%</td></tr><tr><td class='lessen'>125</td><td>30.99 SIGNALLING.UNSPECIFIED</td><td></td><td>19</td><td>0.45</td><td>0.96</td><td>0.506</td><td>0.704</td><td>1.000</td><td>4351</td><td>tags=16%, list=9%, signal=17%</td></tr><tr><td class='lessen'>126</td><td>5.10 FERMENTATION.ALDEHYDE DEHYDROGENASE</td><td></td><td>18</td><td>0.46</td><td>0.96</td><td>0.542</td><td>0.704</td><td>1.000</td><td>5371</td><td>tags=22%, list=11%, signal=25%</td></tr><tr><td class='lessen'>127</td><td>34.3 TRANSPORT.AMINO ACIDS</td><td></td><td>98</td><td>0.33</td><td>0.96</td><td>0.532</td><td>0.699</td><td>1.000</td><td>5643</td><td>tags=26%, list=11%, signal=29%</td></tr><tr><td class='lessen'>128</td><td>17.7.1.2 HORMONE METABOLISM.JASMONATE.SYNTHESIS-DEGRADATION.LIPOXYGENASE</td><td></td><td>32</td><td>0.39</td><td>0.95</td><td>0.535</td><td>0.733</td><td>1.000</td><td>3417</td><td>tags=13%, list=7%, signal=13%</td></tr><tr><td class='lessen'>129</td><td>29.2.2.2 PROTEIN.SYNTHESIS.RIBOSOME BIOGENESIS.ASSEMBLY FACTORS</td><td></td><td>15</td><td>0.47</td><td>0.95</td><td>0.512</td><td>0.738</td><td>1.000</td><td>703</td><td>tags=13%, list=1%, signal=14%</td></tr><tr><td class='lessen'>130</td><td>18.7 CO-FACTOR AND VITAMINE METABOLISM.IRON-SULPHUR CLUSTERS</td><td></td><td>16</td><td>0.45</td><td>0.93</td><td>0.535</td><td>0.793</td><td>1.000</td><td>7141</td><td>tags=31%, list=14%, signal=37%</td></tr><tr><td class='lessen'>131</td><td>27.3.59 RNA.REGULATION OF TRANSCRIPTION.METHYL BINDING DOMAIN PROTEINS</td><td></td><td>26</td><td>0.40</td><td>0.93</td><td>0.547</td><td>0.788</td><td>1.000</td><td>10065</td><td>tags=27%, list=20%, signal=34%</td></tr><tr><td class='lessen'>132</td><td>18 CO-FACTOR AND VITAMINE METABOLISM</td><td></td><td>99</td><td>0.31</td><td>0.93</td><td>0.643</td><td>0.786</td><td>1.000</td><td>10791</td><td>tags=38%, list=22%, signal=49%</td></tr><tr><td class='lessen'>133</td><td>29.5.11.1 PROTEIN.DEGRADATION.UBIQUITIN.UBIQUITIN</td><td></td><td>54</td><td>0.34</td><td>0.92</td><td>0.611</td><td>0.796</td><td>1.000</td><td>6613</td><td>tags=26%, list=13%, signal=30%</td></tr><tr><td class='lessen'>134</td><td>27.3.14 RNA.REGULATION OF TRANSCRIPTION.CCAAT BOX BINDING FACTOR FAMILY, HAP2</td><td></td><td>15</td><td>0.45</td><td>0.92</td><td>0.569</td><td>0.790</td><td>1.000</td><td>12053</td><td>tags=47%, list=24%, signal=62%</td></tr><tr><td class='lessen'>135</td><td>13.2.6.3 AMINO ACID METABOLISM.DEGRADATION.AROMATIC AA.TRYPTOPHAN</td><td></td><td>25</td><td>0.40</td><td>0.91</td><td>0.574</td><td>0.815</td><td>1.000</td><td>3468</td><td>tags=20%, list=7%, signal=22%</td></tr><tr><td class='lessen'>136</td><td>13.1.4.1 AMINO ACID METABOLISM.SYNTHESIS.BRANCHED CHAIN GROUP.COMMON</td><td></td><td>16</td><td>0.44</td><td>0.91</td><td>0.574</td><td>0.818</td><td>1.000</td><td>10133</td><td>tags=31%, list=21%, signal=39%</td></tr><tr><td class='lessen'>137</td><td>30.2.12 SIGNALLING.RECEPTOR KINASES.LEUCINE RICH REPEAT XII</td><td></td><td>234</td><td>0.28</td><td>0.91</td><td>0.805</td><td>0.814</td><td>1.000</td><td>10998</td><td>tags=21%, list=22%, signal=27%</td></tr><tr><td class='lessen'>138</td><td>27.3.26 RNA.REGULATION OF TRANSCRIPTION.MYB-RELATED TRANSCRIPTION FACTOR FAMILY</td><td></td><td>30</td><td>0.38</td><td>0.90</td><td>0.586</td><td>0.830</td><td>1.000</td><td>8375</td><td>tags=33%, list=17%, signal=40%</td></tr><tr><td class='lessen'>139</td><td>27.1.19 RNA.PROCESSING.RIBONUCLEASES</td><td></td><td>95</td><td>0.30</td><td>0.90</td><td>0.714</td><td>0.830</td><td>1.000</td><td>7630</td><td>tags=21%, list=15%, signal=25%</td></tr><tr><td class='lessen'>140</td><td>29.5.9 PROTEIN.DEGRADATION.AAA TYPE</td><td></td><td>59</td><td>0.33</td><td>0.90</td><td>0.677</td><td>0.824</td><td>1.000</td><td>7259</td><td>tags=29%, list=15%, signal=34%</td></tr><tr><td class='lessen'>141</td><td>35.1.3 NOT ASSIGNED.NO ONTOLOGY.ARMADILLO/BETA-CATENIN REPEAT FAMILY PROTEIN</td><td></td><td>39</td><td>0.36</td><td>0.89</td><td>0.605</td><td>0.826</td><td>1.000</td><td>4740</td><td>tags=23%, list=10%, signal=26%</td></tr><tr><td class='lessen'>142</td><td>27.3.21 RNA.REGULATION OF TRANSCRIPTION.GRAS TRANSCRIPTION FACTOR FAMILY</td><td></td><td>40</td><td>0.35</td><td>0.87</td><td>0.695</td><td>0.878</td><td>1.000</td><td>11198</td><td>tags=50%, list=23%, signal=65%</td></tr><tr><td class='lessen'>143</td><td>29.5.11.20 PROTEIN.DEGRADATION.UBIQUITIN.PROTEASOM</td><td></td><td>73</td><td>0.31</td><td>0.87</td><td>0.769</td><td>0.879</td><td>1.000</td><td>12686</td><td>tags=36%, list=26%, signal=48%</td></tr><tr><td class='lessen'>144</td><td>26.22 MISC.SHORT CHAIN DEHYDROGENASE/REDUCTASE (SDR)</td><td></td><td>115</td><td>0.29</td><td>0.87</td><td>0.845</td><td>0.876</td><td>1.000</td><td>6452</td><td>tags=25%, list=13%, signal=29%</td></tr><tr><td class='lessen'>145</td><td>16.2 SECONDARY METABOLISM.PHENYLPROPANOIDS</td><td></td><td>190</td><td>0.27</td><td>0.87</td><td>0.919</td><td>0.878</td><td>1.000</td><td>4853</td><td>tags=11%, list=10%, signal=12%</td></tr><tr><td class='lessen'>146</td><td>26.11.1 MISC.ALCOHOL DEHYDROGENASES.CINNAMYL ALCOHOL DEHYDROGENASE</td><td></td><td>44</td><td>0.33</td><td>0.86</td><td>0.718</td><td>0.893</td><td>1.000</td><td>8828</td><td>tags=30%, list=18%, signal=36%</td></tr><tr><td class='lessen'>147</td><td>26.11 MISC.ALCOHOL DEHYDROGENASES</td><td></td><td>44</td><td>0.33</td><td>0.86</td><td>0.734</td><td>0.890</td><td>1.000</td><td>8828</td><td>tags=30%, list=18%, signal=36%</td></tr><tr><td class='lessen'>148</td><td>31.5 CELL.CELL DEATH</td><td></td><td>55</td><td>0.32</td><td>0.86</td><td>0.753</td><td>0.888</td><td>1.000</td><td>9354</td><td>tags=20%, list=19%, signal=25%</td></tr><tr><td class='lessen'>149</td><td>13.1.5.3.1 AMINO ACID METABOLISM.SYNTHESIS.SERINE-GLYCINE-CYSTEINE GROUP.CYSTEINE.OASTL</td><td></td><td>20</td><td>0.38</td><td>0.85</td><td>0.668</td><td>0.901</td><td>1.000</td><td>11145</td><td>tags=25%, list=23%, signal=32%</td></tr><tr><td class='lessen'>150</td><td>21.2.2 REDOX.ASCORBATE AND GLUTATHIONE.GLUTATHIONE</td><td></td><td>15</td><td>0.41</td><td>0.84</td><td>0.693</td><td>0.905</td><td>1.000</td><td>7028</td><td>tags=53%, list=14%, signal=62%</td></tr><tr><td class='lessen'>151</td><td>11.2.4 LIPID METABOLISM.FA DESATURATION.OMEGA 6 DESATURASE</td><td></td><td>28</td><td>0.36</td><td>0.84</td><td>0.743</td><td>0.913</td><td>1.000</td><td>2482</td><td>tags=11%, list=5%, signal=11%</td></tr><tr><td class='lessen'>152</td><td>29.3.5 PROTEIN.TARGETING.PEROXISOMES</td><td></td><td>23</td><td>0.36</td><td>0.83</td><td>0.771</td><td>0.912</td><td>1.000</td><td>6791</td><td>tags=30%, list=14%, signal=35%</td></tr><tr><td class='lessen'>153</td><td>23.2 NUCLEOTIDE METABOLISM.DEGRADATION</td><td></td><td>50</td><td>0.32</td><td>0.83</td><td>0.827</td><td>0.912</td><td>1.000</td><td>4841</td><td>tags=24%, list=10%, signal=27%</td></tr><tr><td class='lessen'>154</td><td>17.7.1 HORMONE METABOLISM.JASMONATE.SYNTHESIS-DEGRADATION</td><td></td><td>58</td><td>0.30</td><td>0.83</td><td>0.841</td><td>0.908</td><td>1.000</td><td>3417</td><td>tags=10%, list=7%, signal=11%</td></tr><tr><td class='lessen'>155</td><td>11.8 LIPID METABOLISM.EXOTICS (STEROIDS, SQUALENE ETC)</td><td></td><td>95</td><td>0.28</td><td>0.83</td><td>0.931</td><td>0.904</td><td>1.000</td><td>7864</td><td>tags=27%, list=16%, signal=32%</td></tr><tr><td class='lessen'>156</td><td>28.99 DNA.UNSPECIFIED</td><td></td><td>398</td><td>0.24</td><td>0.81</td><td>1.000</td><td>0.930</td><td>1.000</td><td>10873</td><td>tags=15%, list=22%, signal=19%</td></tr><tr><td class='lessen'>157</td><td>8.1 TCA / ORGANIC TRANSFORMATION.TCA</td><td></td><td>52</td><td>0.31</td><td>0.80</td><td>0.850</td><td>0.941</td><td>1.000</td><td>4448</td><td>tags=13%, list=9%, signal=15%</td></tr><tr><td class='lessen'>158</td><td>29.8 PROTEIN.ASSEMBLY AND COFACTOR LIGATION</td><td></td><td>75</td><td>0.28</td><td>0.79</td><td>0.963</td><td>0.966</td><td>1.000</td><td>10983</td><td>tags=17%, list=22%, signal=22%</td></tr><tr><td class='lessen'>159</td><td>9.6 MITOCHONDRIAL ELECTRON TRANSPORT / ATP SYNTHESIS.CYTOCHROME C</td><td></td><td>30</td><td>0.33</td><td>0.76</td><td>0.893</td><td>0.998</td><td>1.000</td><td>8340</td><td>tags=20%, list=17%, signal=24%</td></tr><tr><td class='lessen'>160</td><td>26.27 MISC.CALCINEURIN-LIKE PHOSPHOESTERASE FAMILY PROTEIN</td><td></td><td>18</td><td>0.36</td><td>0.75</td><td>0.836</td><td>1.000</td><td>1.000</td><td>11206</td><td>tags=44%, list=23%, signal=57%</td></tr><tr><td class='lessen'>161</td><td>24 BIODEGRADATION OF XENOBIOTICS</td><td></td><td>35</td><td>0.30</td><td>0.73</td><td>0.947</td><td>1.000</td><td>1.000</td><td>6780</td><td>tags=23%, list=14%, signal=26%</td></tr><tr><td class='lessen'>162</td><td>16.1.2 SECONDARY METABOLISM.ISOPRENOIDS.MEVALONATE PATHWAY</td><td></td><td>47</td><td>0.27</td><td>0.72</td><td>0.984</td><td>1.000</td><td>1.000</td><td>10610</td><td>tags=23%, list=22%, signal=30%</td></tr><tr><td class='lessen'>163</td><td>18.2 CO-FACTOR AND VITAMINE METABOLISM.THIAMINE</td><td></td><td>17</td><td>0.35</td><td>0.72</td><td>0.864</td><td>1.000</td><td>1.000</td><td>5863</td><td>tags=24%, list=12%, signal=27%</td></tr><tr><td class='lessen'>164</td><td>4.3 GLYCOLYSIS.UNCLEAR/DUALLY TARGETED</td><td></td><td>16</td><td>0.35</td><td>0.72</td><td>0.880</td><td>1.000</td><td>1.000</td><td>6490</td><td>tags=19%, list=13%, signal=22%</td></tr><tr><td class='lessen'>165</td><td>29.5.11.5 PROTEIN.DEGRADATION.UBIQUITIN.UBIQUITIN PROTEASE</td><td></td><td>50</td><td>0.27</td><td>0.71</td><td>0.989</td><td>0.999</td><td>1.000</td><td>12472</td><td>tags=34%, list=25%, signal=45%</td></tr><tr><td class='lessen'>166</td><td>13.1.5 AMINO ACID METABOLISM.SYNTHESIS.SERINE-GLYCINE-CYSTEINE GROUP</td><td></td><td>39</td><td>0.28</td><td>0.71</td><td>0.980</td><td>0.995</td><td>1.000</td><td>9836</td><td>tags=31%, list=20%, signal=38%</td></tr><tr><td class='lessen'>167</td><td>13.1.4 AMINO ACID METABOLISM.SYNTHESIS.BRANCHED CHAIN GROUP</td><td></td><td>24</td><td>0.31</td><td>0.71</td><td>0.940</td><td>0.991</td><td>1.000</td><td>5858</td><td>tags=17%, list=12%, signal=19%</td></tr><tr><td class='lessen'>168</td><td>27.3.55 RNA.REGULATION OF TRANSCRIPTION.HDA</td><td></td><td>16</td><td>0.33</td><td>0.69</td><td>0.952</td><td>0.995</td><td>1.000</td><td>1351</td><td>tags=13%, list=3%, signal=13%</td></tr><tr><td class='lessen'>169</td><td>29.2.1.1.1.1 PROTEIN.SYNTHESIS.RIBOSOMAL PROTEIN.PROKARYOTIC.CHLOROPLAST.30S SUBUNIT</td><td></td><td>35</td><td>0.28</td><td>0.69</td><td>0.983</td><td>0.990</td><td>1.000</td><td>10210</td><td>tags=26%, list=21%, signal=32%</td></tr><tr><td class='lessen'>170</td><td>30.2.8.1 SIGNALLING.RECEPTOR KINASES.LEUCINE RICH REPEAT VIII.TYPE 1</td><td></td><td>20</td><td>0.32</td><td>0.69</td><td>0.948</td><td>0.984</td><td>1.000</td><td>4608</td><td>tags=20%, list=9%, signal=22%</td></tr><tr><td class='lessen'>171</td><td>27.1.1 RNA.PROCESSING.SPLICING</td><td></td><td>88</td><td>0.23</td><td>0.67</td><td>1.000</td><td>0.987</td><td>1.000</td><td>11675</td><td>tags=30%, list=24%, signal=39%</td></tr><tr><td class='lessen'>172</td><td>5.3 FERMENTATION.ADH</td><td></td><td>17</td><td>0.31</td><td>0.66</td><td>0.948</td><td>0.986</td><td>1.000</td><td>8475</td><td>tags=29%, list=17%, signal=36%</td></tr><caption class='table'>Table: Gene sets enriched in phenotype <b>PS218_26h (3 samples)<b>&nbsp<a href='gsea_report_for_PS218_26h_1610971023665.tsv'>[plain text format]</a></caption></table></div><br></body></html>