---
title: "AMFinteraction"
author: "zagor"
date: "`r Sys.Date()`"
output:
  html_document:
    toc: yes
    toc_depth: 5
    toc_float:
      collapsed: true
      smooth_scroll: true
    fig_caption: yes
    self_contained: yes
    fig_width: 9
    fig_height: 6
    number_sections: yes
    theme: flatly
    highlight: tango
editor_options: 
  chunk_output_type: console
---
```{r setup, include=FALSE}

knitr::opts_chunk$set(#dev = c('pdf', 'png'),
                      dev = c('png', 'svg', 'pdf'), 
                      fig.align = 'center', 
                      fig.height = 6, 
                      fig.width = 9,
                      warning = FALSE, message = FALSE, echo = TRUE
                      )


```


```{r}

rm(list = ls(all = TRUE))
gc()


```


# packages

```{r, libs}


# %>% 

library(magrittr)
library(ggplot2)
library(data.table)

`%!in%` = Negate(`%in%`)

```

# params

```{r, params}

col = RColorBrewer::brewer.pal(8, 'Paired')[c(3,5,7)]

my_dir = file.path("..", "reports", "AMFinteraction")
if (!dir.exists(my_dir)) {
  dir.create(my_dir)
}

dodge = position_dodge(width = 0.5)
p = rev(col)


```



# input

```{r, input}

fp = file.path('..', 'input')

fn = 'Table_combinedInputs.xlsx'
wb = openxlsx::loadWorkbook(file.path(fp, fn))
sheet_names = names(wb)
sheet_name = grep("AMFinteraction", sheet_names, value = TRUE)

df = openxlsx::read.xlsx(file.path(fp, fn), sheet = sheet_name, startRow = 10)
df = df %>%
  tidyr::pivot_longer(
    cols = c(RiEF, RiMST2, StPT4),       
    names_to = "transcript",                  
    values_to = "Expression"           
  )

df = df[!is.na(df$Expression), ]

df$Genotype = factor(df$Genotype, levels = c('Rywal', 'shPTI5-Rywal L2', 'shPTI5-Rywal L6'))
df$transcript = as.factor(df$transcript)

setDT(df)
df[, Sample.ID := NULL]



```

# scale to mean of NT (Rywal)

```{r, scale}

df.SE = df %>%
  dplyr::group_by(transcript, Genotype) %>%
  dplyr::summarise(
    across(
      where(is.numeric),
      list(mean = mean, sd = sd, se = plotrix::std.error),
      .names = "{.col}_{.fn}"
    ),
    .groups = "drop"
  )

print(df.SE)

df = dplyr::left_join(
  df,
  dplyr::filter(df.SE, Genotype == "Rywal") %>%
    dplyr::select(transcript, NT_mean = Expression_mean),
  by = "transcript"
) %>%
  dplyr::mutate(scaled = Expression / NT_mean)
df[, NT_mean := NULL]


df.SE = df %>%
  dplyr::group_by(transcript, Genotype) %>%
  dplyr::summarise(
    dplyr::across(
      .cols = scaled,
      .fns = list(mean = mean, sd = sd, se = plotrix::std.error, max = max),
      .names = "{.fn}"
    ),
    .groups = "drop"
  )



```


# Pairwise permutation test

```{r, permutation}

set.seed(123456)

mycompare = combn(levels(df$Genotype), 2)
temp = as.data.frame(matrix(nrow = 0, ncol = ncol(mycompare)))
colnames(temp) = combn(levels(df$Genotype), 2, paste0, collapse = ' vs ')

for(i in levels(df$transcript)) {
  
  data = df[df$transcript == i, ]
  data$transcript = droplevels(data$transcript)
  tmp = data %>% dplyr::do(model=as.data.frame(do.call(cbind,
                                                combn(levels(data$Genotype), 2, function(x) {
                                                  dt = data[data$Genotype %in% x, ]
                                                  dt$Genotype = droplevels(dt$Genotype)
                                                  exactRankTests::perm.test(Expression ~ Genotype, 
                                                                            data=dt, 
                                                                            paired=FALSE, 
                                                                            alternative="two.sided",
                                                                            mu=0, exact=TRUE, 
                                                                            conf.int=TRUE, 
                                                                            conf.level=0.95)$p.value
                                                }, simplify = FALSE) -> result)
                                        )
                    )
  
  tmp = t(as.data.frame(unlist(tmp)))
  colnames(tmp) = combn(levels(data$Genotype), 2, paste0, collapse = ' vs ')
  rownames(tmp) = i
  temp = rbind(temp, tmp)
}

temp$transcript = rownames(temp)
perm = tidyr::gather(temp, contrast, p, colnames(temp)[1]:colnames(temp)[ncol(temp)-1], factor_key=TRUE)
perm$group1 = gsub(' vs.*', '', perm$contrast)
perm$group2 = gsub('.* vs ', '', perm$contrast)
perm = perm %>%
  dplyr::left_join(
    df.SE %>% dplyr::select(transcript, Genotype = Genotype, max),
    by = c("transcript", "group2" = "Genotype")
  ) %>%
  dplyr::mutate(y.position = max + 0.5)

# limma style for multiple genes
perm = perm %>%
  dplyr::group_by(contrast) %>%
  dplyr::mutate(p.adj = p.adjust(p, method = "BH")) %>%
  dplyr::ungroup()


perm$stat = ifelse(perm$p.adj >= 0.05, '', ifelse(perm$p.adj >= 0.01, '*', ifelse(perm$p.adj >= 0.001, '**', '***')))
perm$y.position = ifelse(perm$p.adj >= 0.05, NA, perm$y.position) # this will create y.position values that fall outside the plotted y-axis range, or are missing (NA) so R will warn you
mystat_clean = perm %>% dplyr::filter(!is.na(y.position))

write.table(perm, "../output/AMFinteraction_test.txt", sep = "\t", row.names = FALSE, quote = FALSE)



```

# plot with p

```{r, plot}

title = ''

mystat = tibble::as.tibble(
  perm
)

# ggplot() + 
#   theme_void()

myplot = ggplot(data = df, aes(x = Genotype, y = scaled)) +
  geom_boxplot(aes(fill = Genotype), alpha = 0.25) +
  facet_wrap(~transcript) +
  ggtitle(title) +
  theme_classic() +
  scale_colour_manual(name = NULL, values = rev(p)) +
  scale_fill_manual(name = NULL, values = rev(p)) +
  labs(x = '', y = 'Relative gene expression') +
  scale_y_continuous(limits = c(0, NA), breaks = seq(0, max(df$scaled), 1)) +
  theme(
    axis.text.x = element_text(angle = 0, vjust = 1, hjust = 1, size = 12),
    axis.text.y = element_text(angle = 0, vjust = 1, hjust = 1, size = 12),
    axis.title.x = element_text(size = 12, face = "bold"),
    axis.title.y = element_text(angle = 90, size = 12, face = "bold"),
    axis.text = element_text(angle = 0),
    panel.grid.major = element_blank(),
    panel.grid.minor = element_blank(),
    panel.border = element_blank(),
    panel.background = element_blank(),
    legend.position = "bottom",
    legend.background = element_rect(fill = NULL, size = 0.1, linetype = "solid"),
    legend.title = element_text(size = 12, face = "bold"),
    legend.text = element_text(size = 12),
    axis.ticks = element_blank(),
    strip.background = element_blank(),
    strip.text = element_text(size = 14, face = "bold")
  ) +
  guides(fill = "none", shape = "none", colour = "none")




myplot + ggprism::add_pvalue(
  mystat_clean,
  xmin = "group1",
  xmax = "group2",
  y.position = "y.position",
  label = "stat",
  label.size = 5,
  bracket.size = 0.5,
  lineend = "round"
)



myplot = ggplot(df.SE, aes(x = Genotype, y = mean)) + # , pch = genotype)) +
    geom_point(aes(fill = Genotype), size=3.5, shape=22, position = dodge, colour = 'black') +
    geom_point(data = df, 
               aes(x = Genotype, y = scaled, fill = Genotype), #, pch = genotype),
               size = 2.0, shape = 21, colour = 'black',
               position = dodge) +
    geom_errorbar(aes(ymin=mean-se, ymax=mean+se), colour = 'black', width=.3, lwd = 0.5,
                  position = dodge) + 
  facet_wrap(~transcript) + 
    ggtitle('')  + 
    theme_bw() + 
    scale_colour_manual(name = "",
                        values = rev(p)) + 
      scale_fill_manual(name = "",
                        values = rev(p)) +
    # scale_shape_manual(name = "Point",
    #                    values = c(15, 19)) +
            labs(x = '', y = 'Relative gene expression',
                 subtitle = '') +
    scale_y_continuous( limits = c(0, NA),
                        breaks = seq(0, max(df$scaled), 1)
                        ) +
    theme( plot.subtitle = element_text( size = 10 ), 
           axis.text = element_text( size = 12.5 ),
           axis.text.x = element_text( size = 12.5, angle = 90 # 0
                                       ),
           # axis.text.y = element_text( size = 12.5),
           axis.title = element_text( size = 12.5, face = "bold" ),
           title = element_text( size = 15, face = "bold" ),
           axis.ticks.x = element_blank(),
           legend.key.height= unit(1.5, 'cm'),
           legend.key.width= unit(1.75, 'cm'),
           legend.text = element_text(size=12.5),
           legend.title = element_text(size=12.5),
           legend.background = element_rect(fill="transparent", size=.5, linetype="dotted"),
           legend.position="right",
           legend.justification = "right",
           plot.background = element_blank(),
          panel.grid.major = element_blank(),
          panel.grid.minor = element_blank(),#,
          #panel.border = element_blank()
          axis.title.y.right = element_blank(),                # hide right axis title
          axis.text.y.right = element_blank(),                 # hide right axis labels
          axis.ticks.y = element_blank(),                      # hide left/right axis ticks
          axis.text.y = element_text(size = 12.5, margin = margin(r = 0)),  # move left axis labels closer to axis
          panel.spacing = unit(1, "lines"), # panel.spacing = unit(0, "mm"),                       # remove spacing between facets
          # strip.background = element_rect(size = 0.5, fill = "transparent"),         # match default line size of theme_classic
          # strip.background = element_blank() # hide facet outline
          # remove both the panel.border and panel.background to see the axis lines
          panel.border = element_blank(),
          # https://stackoverflow.com/questions/12883386/plotting-only-x-and-y-axis-borders-no-box-in-ggplot2
          axis.line = element_line(),
          # facte box
          # strip.background = element_rect(fill="#FFFFFF"),
          strip.background = element_blank(),
          strip.text = element_text(size = 16, , face = "bold")
           ) +
  guides(colour="none") + 
  guides(fill="none") + 
  theme(legend.position="top") +
  geom_hline(yintercept=1, linetype="dashed", 
                color = "grey60", size=0.5)

myplot + ggprism::add_pvalue(
  mystat_clean,
  xmin = "group1",
  xmax = "group2",
  y.position = "y.position",
  label = "stat",
  label.size = 5,
  bracket.size = 0.5,
  lineend = "round"
)


ggsave(
  filename = paste0(my_dir, '/fig1', '.pdf'),
  plot = last_plot(),
  device = pdf,
  path = NULL,
  scale = 1,
  width = 4,
  height = 7,
  units = c("in"),
  dpi = 300,
  limitsize = TRUE,
  bg = NULL
)
ggsave(
  filename = paste0(my_dir, '/fig1', '.svg'),
  plot = last_plot(),
  device = svg,
  path = NULL,
  scale = 1,
  width = 4,
  height = 7,
  units = c("in"),
  dpi = 300,
  limitsize = TRUE,
  bg = NULL
)


```


# session info

```{r, sessionInfo}

sessionInfo()

```

